<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.36">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Sampling oms</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting-01c78b5cd655e4cd89133cf59d535862.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap-11df3c80491ea81009d83f04e2a62a23.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<link href="index_files/libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="index_files/libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="index_files/libs/vis-9.1.0/vis-network.min.css" rel="stylesheet">
<script src="index_files/libs/vis-9.1.0/vis-network.min.js"></script>
<script src="index_files/libs/visNetwork-binding-2.1.2/visNetwork.js"></script>
<script src="index_files/libs/viz-1.8.2/viz.js"></script>
<link href="index_files/libs/DiagrammeR-styles-0.2/styles.css" rel="stylesheet">
<script src="index_files/libs/grViz-binding-1.0.11/grViz.js"></script>


</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#project-and-data-vizualisation" id="toc-project-and-data-vizualisation" class="nav-link active" data-scroll-target="#project-and-data-vizualisation">Project and data vizualisation</a></li>
  <li><a href="#what-is-the-best-proxy-for-observational-mating-success" id="toc-what-is-the-best-proxy-for-observational-mating-success" class="nav-link" data-scroll-target="#what-is-the-best-proxy-for-observational-mating-success">What is the best proxy for observational mating success?</a>
  <ul class="collapse">
  <li><a href="#visit-sequences" id="toc-visit-sequences" class="nav-link" data-scroll-target="#visit-sequences">Visit sequences</a>
  <ul class="collapse">
  <li><a href="#acp" id="toc-acp" class="nav-link" data-scroll-target="#acp">ACP</a></li>
  <li><a href="#correlation-test" id="toc-correlation-test" class="nav-link" data-scroll-target="#correlation-test">Correlation test</a></li>
  <li><a href="#linear-models-and-r2" id="toc-linear-models-and-r2" class="nav-link" data-scroll-target="#linear-models-and-r2">Linear models and R<sup>2</sup></a></li>
  <li><a href="#aic" id="toc-aic" class="nav-link" data-scroll-target="#aic">AIC</a></li>
  <li><a href="#root-mean-squared-error" id="toc-root-mean-squared-error" class="nav-link" data-scroll-target="#root-mean-squared-error">Root Mean Squared Error</a></li>
  <li><a href="#conclusions" id="toc-conclusions" class="nav-link" data-scroll-target="#conclusions">Conclusions</a></li>
  </ul></li>
  <li><a href="#which-traits-are-good-predictor-of-mate-access" id="toc-which-traits-are-good-predictor-of-mate-access" class="nav-link" data-scroll-target="#which-traits-are-good-predictor-of-mate-access">Which traits are good predictor of mate access?</a>
  <ul class="collapse">
  <li><a href="#simple-linear-models" id="toc-simple-linear-models" class="nav-link" data-scroll-target="#simple-linear-models">Simple linear models</a></li>
  <li><a href="#sem" id="toc-sem" class="nav-link" data-scroll-target="#sem">SEM</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#comparing-observational-to-genetic-mating-success" id="toc-comparing-observational-to-genetic-mating-success" class="nav-link" data-scroll-target="#comparing-observational-to-genetic-mating-success">Comparing observational to genetic mating success</a>
  <ul class="collapse">
  <li><a href="#true-batemans-gradient-on-all-data-based-on-observations" id="toc-true-batemans-gradient-on-all-data-based-on-observations" class="nav-link" data-scroll-target="#true-batemans-gradient-on-all-data-based-on-observations">True Bateman’s gradient on all data based on observations</a></li>
  <li><a href="#bateman-hedges" id="toc-bateman-hedges" class="nav-link" data-scroll-target="#bateman-hedges">Bateman Hedge’s</a>
  <ul class="collapse">
  <li><a href="#per-session" id="toc-per-session" class="nav-link" data-scroll-target="#per-session">Per session:</a></li>
  <li><a href="#overall" id="toc-overall" class="nav-link" data-scroll-target="#overall">Overall:</a></li>
  </ul></li>
  <li><a href="#variance-decomposition" id="toc-variance-decomposition" class="nav-link" data-scroll-target="#variance-decomposition">Variance decomposition</a></li>
  </ul></li>
  </ul>
</nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Sampling oms</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="project-and-data-vizualisation" class="level1">
<h1>Project and data vizualisation</h1>
<p>Visualize the pipeline analyses:</p>
<div class="cell">
<div class="cell-output-display">
<div class="visNetwork html-widget html-fill-item" id="htmlwidget-b8f16dc07f3dd3d59cc7" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-b8f16dc07f3dd3d59cc7">{"x":{"nodes":{"name":["data_contact","data_genotypes","data_hedges_false","data_hedges_true","data_id","data_obs","data_proxy","data_sampling","data_sampling_bateman","data_sampling_rs_ms","data_session","data_true_bateman","data_true_rs_ms","index","pcas_10","pcas_20","pcas_5","get_data_contact","load_data_id","get_sampling_bateman","load_data_obs","get_data_session","get_sampling_rs_ms","get_true_rs_ms","load_data","get_hedges","get_pca","compute_oms_and_proxy","get_true_bateman"],"type":["stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","function","function","function","function","function","function","function","function","function","function","function","function"],"description":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"status":["uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","dispatched","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate"],"seconds":[0.946,0.003,0.029,0.11,0.059,0.011,0.039,11.549,687.8630000000001,2.421,0.037,1.712,0.106,7.967,0.259,0.266,0.902,null,null,null,null,null,null,null,null,null,null,null,null],"bytes":[18013,13324,531220,573957,3250,49381,8661,63177025,7627687,9587597,558,2605,2195,5115909,334680,334947,334278,null,null,null,null,null,null,null,null,null,null,null,null],"branches":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"label":["data_contact","data_genotypes","data_hedges_false","data_hedges_true","data_id","data_obs","data_proxy","data_sampling","data_sampling_bateman","data_sampling_rs_ms","data_session","data_true_bateman","data_true_rs_ms","index","pcas_10","pcas_20","pcas_5","get_data_contact","load_data_id","get_sampling_bateman","load_data_obs","get_data_session","get_sampling_rs_ms","get_true_rs_ms","load_data","get_hedges","get_pca","compute_oms_and_proxy","get_true_bateman"],"color":["#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#DC863B","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823"],"id":["data_contact","data_genotypes","data_hedges_false","data_hedges_true","data_id","data_obs","data_proxy","data_sampling","data_sampling_bateman","data_sampling_rs_ms","data_session","data_true_bateman","data_true_rs_ms","index","pcas_10","pcas_20","pcas_5","get_data_contact","load_data_id","get_sampling_bateman","load_data_obs","get_data_session","get_sampling_rs_ms","get_true_rs_ms","load_data","get_hedges","get_pca","compute_oms_and_proxy","get_true_bateman"],"level":[3,2,7,7,2,2,5,2,3,2,4,6,3,8,6,6,6,1,1,1,1,1,0,1,1,1,1,1,1],"shape":["dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle"]},"edges":{"from":["data_proxy","get_pca","data_sampling_bateman","data_true_bateman","get_hedges","data_proxy","get_pca","data_contact","get_data_session","load_data","load_data_obs","data_sampling_bateman","data_true_bateman","get_hedges","data_sampling_rs_ms","get_sampling_bateman","load_data","data_proxy","data_true_rs_ms","get_true_bateman","load_data","data_genotypes","data_id","get_true_rs_ms","data_proxy","get_pca","data_obs","get_data_contact","data_contact","data_hedges_false","data_hedges_true","data_id","data_obs","data_proxy","data_session","data_true_bateman","pcas_10","pcas_20","pcas_5","load_data_id","compute_oms_and_proxy","data_contact","data_id","data_session"],"to":["pcas_5","pcas_5","data_hedges_true","data_hedges_true","data_hedges_true","pcas_20","pcas_20","data_session","data_session","data_sampling","data_obs","data_hedges_false","data_hedges_false","data_hedges_false","data_sampling_bateman","data_sampling_bateman","data_sampling_rs_ms","data_true_bateman","data_true_bateman","data_true_bateman","data_genotypes","data_true_rs_ms","data_true_rs_ms","data_true_rs_ms","pcas_10","pcas_10","data_contact","data_contact","index","index","index","index","index","index","index","index","index","index","index","data_id","data_proxy","data_proxy","data_proxy","data_proxy"],"arrows":["to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot","physics":false},"manipulation":{"enabled":false},"edges":{"smooth":{"type":"cubicBezier","forceDirection":"horizontal"}},"physics":{"stabilization":false},"interaction":{"zoomSpeed":1},"layout":{"hierarchical":{"enabled":true,"direction":"LR"}}},"groups":null,"width":null,"height":null,"idselection":{"enabled":false,"style":"width: 150px; height: 26px","useLabels":true,"main":"Select by id"},"byselection":{"enabled":false,"style":"width: 150px; height: 26px","multiple":false,"hideColor":"rgba(200,200,200,0.5)","highlight":false},"main":{"text":"","style":"font-family:Georgia, Times New Roman, Times, serif;font-weight:bold;font-size:20px;text-align:center;"},"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","highlight":{"enabled":true,"hoverNearest":false,"degree":{"from":1,"to":1},"algorithm":"hierarchical","hideColor":"rgba(200,200,200,0.5)","labelOnly":true},"collapse":{"enabled":true,"fit":false,"resetHighlight":true,"clusterOptions":null,"keepCoord":true,"labelSuffix":"(cluster)"},"legend":{"width":0.2,"useGroups":false,"position":"right","ncol":1,"stepX":100,"stepY":100,"zoom":true,"nodes":{"label":["Up to date","Dispatched","Stem","Function"],"color":["#354823","#DC863B","#899DA4","#899DA4"],"shape":["dot","dot","dot","triangle"]},"nodesToDataframe":true},"tooltipStay":300,"tooltipStyle":"position: fixed;visibility:hidden;padding: 5px;white-space: nowrap;font-family: verdana;font-size:14px;font-color:#000000;background-color: #f5f4ed;-moz-border-radius: 3px;-webkit-border-radius: 3px;border-radius: 3px;border: 1px solid #808074;box-shadow: 3px 3px 10px rgba(0, 0, 0, 0.2);"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 4%">
<col style="width: 5%">
<col style="width: 7%">
<col style="width: 4%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 8%">
<col style="width: 6%">
<col style="width: 7%">
<col style="width: 7%">
<col style="width: 6%">
<col style="width: 7%">
<col style="width: 7%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">session</th>
<th style="text-align: left;">id</th>
<th style="text-align: right;">sr_fem_total</th>
<th style="text-align: right;">nb_flo</th>
<th style="text-align: right;">nb_flo_open</th>
<th style="text-align: right;">nb_flo_all</th>
<th style="text-align: right;">height_max</th>
<th style="text-align: right;">height_mean</th>
<th style="text-align: right;">nb_poll_focal</th>
<th style="text-align: right;">oms_fem_co5</th>
<th style="text-align: right;">oms_fem_co10</th>
<th style="text-align: right;">oms_fem_co20</th>
<th style="text-align: right;">oms_mal_co5</th>
<th style="text-align: right;">oms_mal_co10</th>
<th style="text-align: right;">oms_mal_co20</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">56</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">20</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">40.4</td>
<td style="text-align: right;">36.73333</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.02</td>
<td style="text-align: right;">119</td>
<td style="text-align: right;">19</td>
<td style="text-align: right;">21</td>
<td style="text-align: right;">26</td>
<td style="text-align: right;">31.5</td>
<td style="text-align: right;">24.40000</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.03</td>
<td style="text-align: right;">38</td>
<td style="text-align: right;">11</td>
<td style="text-align: right;">13</td>
<td style="text-align: right;">13</td>
<td style="text-align: right;">37.4</td>
<td style="text-align: right;">35.45000</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.04</td>
<td style="text-align: right;">169</td>
<td style="text-align: right;">19</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">28</td>
<td style="text-align: right;">36.4</td>
<td style="text-align: right;">33.33333</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.05</td>
<td style="text-align: right;">29</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">18</td>
<td style="text-align: right;">36.5</td>
<td style="text-align: right;">31.63333</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.06</td>
<td style="text-align: right;">130</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">22</td>
<td style="text-align: right;">30</td>
<td style="text-align: right;">40.3</td>
<td style="text-align: right;">34.35000</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
</tr>
</tbody>
</table>
</div>
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 9%">
<col style="width: 8%">
<col style="width: 8%">
<col style="width: 11%">
<col style="width: 11%">
<col style="width: 3%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 9%">
<col style="width: 17%">
<col style="width: 7%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">session</th>
<th style="text-align: left;">people</th>
<th style="text-align: right;">time</th>
<th style="text-align: right;">duration</th>
<th style="text-align: left;">ID_full</th>
<th style="text-align: right;">ID</th>
<th style="text-align: right;">noHp</th>
<th style="text-align: left;">flow</th>
<th style="text-align: left;">id_flow</th>
<th style="text-align: left;">id_flow_full</th>
<th style="text-align: left;">ttt</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: right;">32.768</td>
<td style="text-align: right;">2.815</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">3</td>
<td style="text-align: left;">1_1_3</td>
<td style="text-align: left;">1.FA1.01_1_3</td>
<td style="text-align: left;">1_low</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: right;">36.822</td>
<td style="text-align: right;">3.327</td>
<td style="text-align: left;">1.FA1.06</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">2</td>
<td style="text-align: left;">NR</td>
<td style="text-align: left;">6_2_NR</td>
<td style="text-align: left;">1.FA1.06_2_NR</td>
<td style="text-align: left;">1_low</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: right;">56.448</td>
<td style="text-align: right;">8.191</td>
<td style="text-align: left;">1.FA1.03</td>
<td style="text-align: right;">3</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">8</td>
<td style="text-align: left;">3_1_8</td>
<td style="text-align: left;">1.FA1.03_1_8</td>
<td style="text-align: left;">1_low</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: right;">67.371</td>
<td style="text-align: right;">12.628</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">NR</td>
<td style="text-align: left;">1_1_NR</td>
<td style="text-align: left;">1.FA1.01_1_NR</td>
<td style="text-align: left;">1_low</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: right;">80.000</td>
<td style="text-align: right;">1.023</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">3</td>
<td style="text-align: left;">1_1_3</td>
<td style="text-align: left;">1.FA1.01_1_3</td>
<td style="text-align: left;">1_low</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: right;">81.024</td>
<td style="text-align: right;">28.650</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">4</td>
<td style="text-align: left;">1_1_4</td>
<td style="text-align: left;">1.FA1.01_1_4</td>
<td style="text-align: left;">1_low</td>
</tr>
</tbody>
</table>
</div>
</div>
</section>
<section id="what-is-the-best-proxy-for-observational-mating-success" class="level1">
<h1>What is the best proxy for observational mating success?</h1>
<section id="visit-sequences" class="level2">
<h2 class="anchored" data-anchor-id="visit-sequences">Visit sequences</h2>
<p>We have conducted an experiment in which we surveyed each pollinator one by one along the visit sequences, up to the flower scale. By making assumptions about the carry-over (i.e., the number of successive flower visits during which pollen export remains effective), we are able to estimate the number of observational sexual partners, both for the female and the male function. We assume that the carry-over is about 10, based on a previous study on the sister species <em>Brassica napus</em>: for the male function, 91% of the seeds were sired in the first four flowers visited after a focal individual, and no seeds were sired after the 14th flower visited <span class="citation" data-cites="cresswell_method_1994">[<a href="#ref-cresswell_method_1994" role="doc-biblioref">1</a>]</span>.</p>
<p>Now, we imagine a more common experiment <em>in natura</em> in which we cannot follow the pollinators one by one. We could rather observe plant by plant. We then have the following data :</p>
<ul>
<li>the number of arrivals (female function) / departures (male function) per individual, which are highly correlated = number of contacts (which may be independent or not) per individual<br>
</li>
<li>the total number of visits per individual<br>
</li>
<li>the total visit duration per individual (but not useful here, as we only formulate hypotheses about visit sequences, without getting interested in the number of pollen grains exported/deposited): we leave that aside</li>
</ul>
<p>We can imagine that each observer notes the time of the pollinator’s arrival on each plant (start), to order events of contacts:</p>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 8%">
<col style="width: 7%">
<col style="width: 10%">
<col style="width: 22%">
<col style="width: 8%">
<col style="width: 10%">
<col style="width: 10%">
<col style="width: 21%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">session</th>
<th style="text-align: left;">people</th>
<th style="text-align: left;">id</th>
<th style="text-align: right;">consecutive_contact</th>
<th style="text-align: right;">start</th>
<th style="text-align: right;">nb_visit</th>
<th style="text-align: right;">duration</th>
<th style="text-align: right;">no_contact_with_id</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">32.768</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">2.815</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: left;">1.FA1.06</td>
<td style="text-align: right;">2</td>
<td style="text-align: right;">36.822</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">3.327</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: left;">1.FA1.03</td>
<td style="text-align: right;">3</td>
<td style="text-align: right;">56.448</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8.191</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">4</td>
<td style="text-align: right;">67.371</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">105.677</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: left;">1.FA1.06</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">183.382</td>
<td style="text-align: right;">2</td>
<td style="text-align: right;">10.877</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">201.195</td>
<td style="text-align: right;">2</td>
<td style="text-align: right;">13.056</td>
<td style="text-align: right;">3</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>But then, what to do with this? We don’t know the identity of the pollinator,<br>
so it is difficult to infer anything from the visit order…</p>
<p>Something more realistic is to start from data at the individual level without inferring anything about the effect of visit order:</p>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;">session</th>
<th style="text-align: left;">id</th>
<th style="text-align: right;">nb_contact</th>
<th style="text-align: right;">nb_visit</th>
<th style="text-align: right;">dur_visit</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">25</td>
<td style="text-align: right;">213.846</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.02</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">76.402</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.03</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">17</td>
<td style="text-align: right;">84.098</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.04</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">118.506</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.05</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">65.717</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.06</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">29</td>
<td style="text-align: right;">157.325</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>To discuss: something smart to do with contact order?</p>
<p>Then, we can summarize data at the scale of the observation session. At least useful to compare<br>
populations between them? We can get the following data:</p>
<ul>
<li>Total number of contacts with id</li>
<li>Total number of visits<br>
</li>
<li>Total number of visited individuals (maximum oms)<br>
</li>
<li>Total number of pollinators (but see below)<br>
</li>
<li>Total visit duration (but again, we leave that aside)</li>
</ul>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 10%">
<col style="width: 10%">
<col style="width: 22%">
<col style="width: 19%">
<col style="width: 19%">
<col style="width: 18%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">session</th>
<th style="text-align: right;">max_oms</th>
<th style="text-align: right;">total_nb_contact</th>
<th style="text-align: right;">total_nb_visit</th>
<th style="text-align: right;">total_duration</th>
<th style="text-align: right;">total_nb_poll</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA2</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">27</td>
<td style="text-align: right;">124</td>
<td style="text-align: right;">1098.092</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FO1</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">62</td>
<td style="text-align: right;">486</td>
<td style="text-align: right;">3642.479</td>
<td style="text-align: right;">4</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FO2</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">42</td>
<td style="text-align: right;">373</td>
<td style="text-align: right;">3178.549</td>
<td style="text-align: right;">3</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.MO1</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">50</td>
<td style="text-align: right;">185</td>
<td style="text-align: right;">1509.840</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.MO2</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">11</td>
<td style="text-align: right;">109</td>
<td style="text-align: right;">1463.275</td>
<td style="text-align: right;">2</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Because, imagine that one individual had a lot of contacts with pollinators, but<br>
that only a few other individuals were visited: this should weaken the link between<br>
the number of contacts per id and oms.</p>
<p>Using the data table with contacts ordered by time, we could imagine estimating<br>
the diversity of potential sexual partners according to their relative frequency<br>
(e.g., using Hill/Shannon/Simpson indices).</p>
<p>NB: In our experiment, we know for sure the number of pollinators that visited<br>
plants. <em>In natura</em>, it would be much more difficult, but we could at least get an estimation<br>
based on the species diversity that we observed (i.e., if we observed<br>
two contacts by <em>Bombus</em> and one contact by <em>Apis</em>, there are at least two different<br>
pollinators).</p>
<p>We now investigate which proxy best correlates with the observational number of mates.<br>
We estimate the latter using a carry-over of 10, as well as 5 and 20. We thus<br>
have two main proxies:</p>
<ul>
<li><ol type="A">
<li>the number of contacts per individual<br>
</li>
</ol></li>
<li><ol start="2" type="A">
<li>the total number of visits per individual</li>
</ol></li>
</ul>
<ol type="A">
<li>For the number of contacts per individual, we can explore its correlation with<br>
oms using:<br>
</li>
</ol>
<ul>
<li>index_A1: the raw number of contacts per individual<br>
</li>
<li>index_A2: relativized by the total number of contacts in the session<br>
</li>
<li>index_A3: multiplied by the total number of potential sexual partners<br>
</li>
<li>index_A4: combining index_A2 and index_A3<br>
</li>
<li>index_A5 (optional, to discuss): index_A4 divided by the total number of pollinators</li>
</ul>
<ol start="2" type="A">
<li>For the total number of visits per individual:<br>
</li>
</ol>
<ul>
<li>index_B1: the raw number of visits per individual<br>
</li>
<li>index_B2: relativized by the total number of visits in the session<br>
</li>
<li>index_B3: multiplied by the total number of potential sexual partners<br>
</li>
<li>index_B4: combining index_B2 and index_B3<br>
</li>
<li>index_B5 (optional, to discuss): index_B4 divided by the total number of pollinators</li>
</ul>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 3%">
<col style="width: 1%">
<col style="width: 3%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 4%">
<col style="width: 4%">
<col style="width: 4%">
<col style="width: 3%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">session</th>
<th style="text-align: left;">id</th>
<th style="text-align: right;">sr_fem_total</th>
<th style="text-align: right;">nb_flo</th>
<th style="text-align: right;">nb_flo_open</th>
<th style="text-align: right;">nb_flo_all</th>
<th style="text-align: right;">height_max</th>
<th style="text-align: right;">height_mean</th>
<th style="text-align: right;">nb_poll_focal</th>
<th style="text-align: right;">oms_fem_co5</th>
<th style="text-align: right;">oms_fem_co10</th>
<th style="text-align: right;">oms_fem_co20</th>
<th style="text-align: right;">oms_mal_co5</th>
<th style="text-align: right;">oms_mal_co10</th>
<th style="text-align: right;">oms_mal_co20</th>
<th style="text-align: right;">nb_contact</th>
<th style="text-align: right;">nb_visit</th>
<th style="text-align: right;">dur_visit</th>
<th style="text-align: right;">max_oms</th>
<th style="text-align: right;">total_nb_contact</th>
<th style="text-align: right;">total_nb_visit</th>
<th style="text-align: right;">total_duration</th>
<th style="text-align: right;">total_nb_poll</th>
<th style="text-align: right;">index_A1</th>
<th style="text-align: right;">index_A2</th>
<th style="text-align: right;">index_A3</th>
<th style="text-align: right;">index_A4</th>
<th style="text-align: right;">index_A5</th>
<th style="text-align: right;">index_B1</th>
<th style="text-align: right;">index_B2</th>
<th style="text-align: right;">index_B3</th>
<th style="text-align: right;">index_B4</th>
<th style="text-align: right;">index_B5</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">56</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">20</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">40.4</td>
<td style="text-align: right;">36.73333</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">25</td>
<td style="text-align: right;">213.846</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">0.1369863</td>
<td style="text-align: right;">100</td>
<td style="text-align: right;">1.3698630</td>
<td style="text-align: right;">1.3698630</td>
<td style="text-align: right;">25</td>
<td style="text-align: right;">0.1269036</td>
<td style="text-align: right;">250</td>
<td style="text-align: right;">1.2690355</td>
<td style="text-align: right;">1.3698630</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.02</td>
<td style="text-align: right;">119</td>
<td style="text-align: right;">19</td>
<td style="text-align: right;">21</td>
<td style="text-align: right;">26</td>
<td style="text-align: right;">31.5</td>
<td style="text-align: right;">24.40000</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">76.402</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">0.0684932</td>
<td style="text-align: right;">50</td>
<td style="text-align: right;">0.6849315</td>
<td style="text-align: right;">0.6849315</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">0.0710660</td>
<td style="text-align: right;">140</td>
<td style="text-align: right;">0.7106599</td>
<td style="text-align: right;">0.6849315</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.03</td>
<td style="text-align: right;">38</td>
<td style="text-align: right;">11</td>
<td style="text-align: right;">13</td>
<td style="text-align: right;">13</td>
<td style="text-align: right;">37.4</td>
<td style="text-align: right;">35.45000</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">17</td>
<td style="text-align: right;">84.098</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">0.1095890</td>
<td style="text-align: right;">80</td>
<td style="text-align: right;">1.0958904</td>
<td style="text-align: right;">1.0958904</td>
<td style="text-align: right;">17</td>
<td style="text-align: right;">0.0862944</td>
<td style="text-align: right;">170</td>
<td style="text-align: right;">0.8629442</td>
<td style="text-align: right;">1.0958904</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.04</td>
<td style="text-align: right;">169</td>
<td style="text-align: right;">19</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">28</td>
<td style="text-align: right;">36.4</td>
<td style="text-align: right;">33.33333</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">118.506</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">0.1095890</td>
<td style="text-align: right;">80</td>
<td style="text-align: right;">1.0958904</td>
<td style="text-align: right;">1.0958904</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">0.1167513</td>
<td style="text-align: right;">230</td>
<td style="text-align: right;">1.1675127</td>
<td style="text-align: right;">1.0958904</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.05</td>
<td style="text-align: right;">29</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">18</td>
<td style="text-align: right;">36.5</td>
<td style="text-align: right;">31.63333</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">65.717</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">0.0821918</td>
<td style="text-align: right;">60</td>
<td style="text-align: right;">0.8219178</td>
<td style="text-align: right;">0.8219178</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">0.0609137</td>
<td style="text-align: right;">120</td>
<td style="text-align: right;">0.6091371</td>
<td style="text-align: right;">0.8219178</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.06</td>
<td style="text-align: right;">130</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">22</td>
<td style="text-align: right;">30</td>
<td style="text-align: right;">40.3</td>
<td style="text-align: right;">34.35000</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">29</td>
<td style="text-align: right;">157.325</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">0.1095890</td>
<td style="text-align: right;">80</td>
<td style="text-align: right;">1.0958904</td>
<td style="text-align: right;">1.0958904</td>
<td style="text-align: right;">29</td>
<td style="text-align: right;">0.1472081</td>
<td style="text-align: right;">290</td>
<td style="text-align: right;">1.4720812</td>
<td style="text-align: right;">1.0958904</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Okay now we proceed to ACP.</p>
<section id="acp" class="level3">
<h3 class="anchored" data-anchor-id="acp">ACP</h3>
<div class="grid">
<section id="females" class="level4 g-col-6">
<h4 class="anchored" data-anchor-id="females">Females</h4>
<p><strong>Carry-over 10</strong></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-7-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><strong>Carry-over 5</strong></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-8-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><strong>Carry-over 20</strong></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="males" class="level4 g-col-6">
<h4 class="anchored" data-anchor-id="males">Males</h4>
<p><strong>Carry-over 10</strong></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-10-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><strong>Carry-over 5</strong></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><strong>Carry-over 20</strong></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-12-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
</div>
<blockquote class="blockquote">
<p><strong>The three best based on PCA : index_A3, index_B3 and index A_1. From now on we focus solely on a carry-over of 10, as it’s very similar.</strong></p>
</blockquote>
</section>
<section id="correlation-test" class="level3">
<h3 class="anchored" data-anchor-id="correlation-test">Correlation test</h3>
<div class="grid">
<section id="females-1" class="level4 g-col-6">
<h4 class="anchored" data-anchor-id="females-1">Females</h4>
<p><strong>A1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_fem_co10 and data_proxy$index_A1
t = 11.381, df = 136, p-value &lt; 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.6015364 0.7750788
sample estimates:
      cor 
0.6984348 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_fem_co10 and data_proxy$index_A1
S = 109846, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.7492027 </code></pre>
</div>
</div>
<p><strong>A2</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_fem_co10 and data_proxy$index_A2
t = -0.52347, df = 136, p-value = 0.6015
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.2103704  0.1231860
sample estimates:
        cor 
-0.04484194 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_fem_co10 and data_proxy$index_A2
S = 395711, p-value = 0.2601
alternative hypothesis: true rho is not equal to 0
sample estimates:
       rho 
0.09652734 </code></pre>
</div>
</div>
<p><strong>A3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_fem_co10 and data_proxy$index_A3
t = 13.653, df = 136, p-value &lt; 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.6796225 0.8229165
sample estimates:
      cor 
0.7603731 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_fem_co10 and data_proxy$index_A3
S = 84479, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.8071208 </code></pre>
</div>
</div>
<p><strong>A4</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_fem_co10 and data_proxy$index_A4
t = 3.2682, df = 136, p-value = 0.00137
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.1075987 0.4181027
sample estimates:
      cor 
0.2698515 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_fem_co10 and data_proxy$index_A4
S = 288311, p-value = 4.092e-05
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.3417384 </code></pre>
</div>
</div>
<p><strong>A5</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_fem_co10 and data_proxy$index_A5
t = -1.9169, df = 136, p-value = 0.05734
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.320614804  0.005041439
sample estimates:
    cor 
-0.1622 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_fem_co10 and data_proxy$index_A5
S = 463225, p-value = 0.5021
alternative hypothesis: true rho is not equal to 0
sample estimates:
        rho 
-0.05761802 </code></pre>
</div>
</div>
<p><strong>B1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_fem_co10 and data_proxy$index_B1
t = 6.6539, df = 136, p-value = 6.414e-10
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.3581291 0.6119994
sample estimates:
      cor 
0.4955762 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_fem_co10 and data_proxy$index_B1
S = 194058, p-value = 1.306e-12
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.5569332 </code></pre>
</div>
</div>
<p><strong>B2</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_fem_co10 and data_proxy$index_B2
t = -0.25705, df = 136, p-value = 0.7975
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.1884475  0.1456044
sample estimates:
       cor 
-0.0220366 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_fem_co10 and data_proxy$index_B2
S = 374554, p-value = 0.0901
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.1448333 </code></pre>
</div>
</div>
<p><strong>B3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_fem_co10 and data_proxy$index_B3
t = 7.394, df = 136, p-value = 1.317e-11
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.4045660 0.6448723
sample estimates:
      cor 
0.5354704 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_fem_co10 and data_proxy$index_B3
S = 160442, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.6336852 </code></pre>
</div>
</div>
<p><strong>B4</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_fem_co10 and data_proxy$index_B4
t = 2.2365, df = 136, p-value = 0.02695
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.02193452 0.34460763
sample estimates:
     cor 
0.188349 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_fem_co10 and data_proxy$index_B4
S = 309817, p-value = 0.0004958
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.2926366 </code></pre>
</div>
</div>
<p><strong>B5</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_fem_co10 and data_proxy$index_B5
t = -1.9169, df = 136, p-value = 0.05734
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.320614804  0.005041439
sample estimates:
    cor 
-0.1622 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_fem_co10 and data_proxy$index_B5
S = 463225, p-value = 0.5021
alternative hypothesis: true rho is not equal to 0
sample estimates:
        rho 
-0.05761802 </code></pre>
</div>
</div>
</section>
<section id="males-1" class="level4 g-col-6">
<h4 class="anchored" data-anchor-id="males-1">Males</h4>
<p><strong>A1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_mal_co10 and data_proxy$index_A1
t = 12.122, df = 136, p-value &lt; 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.6293332 0.7923391
sample estimates:
      cor 
0.7206511 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_mal_co10 and data_proxy$index_A1
S = 104123, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
    rho 
0.76227 </code></pre>
</div>
</div>
<p><strong>A2</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_mal_co10 and data_proxy$index_A2
t = -0.16192, df = 136, p-value = 0.8716
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.1805694  0.1535777
sample estimates:
        cor 
-0.01388345 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_mal_co10 and data_proxy$index_A2
S = 398429, p-value = 0.2921
alternative hypothesis: true rho is not equal to 0
sample estimates:
       rho 
0.09032237 </code></pre>
</div>
</div>
<p><strong>A3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_mal_co10 and data_proxy$index_A3
t = 14.541, df = 136, p-value &lt; 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.7049039 0.8379812
sample estimates:
      cor 
0.7801166 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_mal_co10 and data_proxy$index_A3
S = 77512, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.8230266 </code></pre>
</div>
</div>
<p><strong>A4</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_mal_co10 and data_proxy$index_A4
t = 3.6489, df = 136, p-value = 0.0003743
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.1384167 0.4435846
sample estimates:
      cor 
0.2986146 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_mal_co10 and data_proxy$index_A4
S = 283390, p-value = 2.173e-05
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.3529735 </code></pre>
</div>
</div>
<p><strong>A5</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_mal_co10 and data_proxy$index_A5
t = -1.8643, df = 136, p-value = 0.06444
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.316610047  0.009498434
sample estimates:
      cor 
-0.157857 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_mal_co10 and data_proxy$index_A5
S = 470381, p-value = 0.3886
alternative hypothesis: true rho is not equal to 0
sample estimates:
        rho 
-0.07395657 </code></pre>
</div>
</div>
<p><strong>B1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_mal_co10 and data_proxy$index_B1
t = 6.6886, df = 136, p-value = 5.368e-10
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.3603790 0.6136126
sample estimates:
      cor 
0.4975225 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_mal_co10 and data_proxy$index_B1
S = 195666, p-value = 1.963e-12
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.5532618 </code></pre>
</div>
</div>
<p><strong>B2</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_mal_co10 and data_proxy$index_B2
t = -0.35112, df = 136, p-value = 0.726
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.1962127  0.1377027
sample estimates:
        cor 
-0.03009462 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_mal_co10 and data_proxy$index_B2
S = 386699, p-value = 0.1714
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.1171035 </code></pre>
</div>
</div>
<p><strong>B3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_mal_co10 and data_proxy$index_B3
t = 7.4052, df = 136, p-value = 1.239e-11
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.4052485 0.6453488
sample estimates:
      cor 
0.5360524 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_mal_co10 and data_proxy$index_B3
S = 160222, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.6341882 </code></pre>
</div>
</div>
<p><strong>B4</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_mal_co10 and data_proxy$index_B4
t = 2.1615, df = 136, p-value = 0.03241
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.0156129 0.3390227
sample estimates:
      cor 
0.1822423 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_mal_co10 and data_proxy$index_B4
S = 319974, p-value = 0.001394
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.2694464 </code></pre>
</div>
</div>
<p><strong>B5</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$oms_mal_co10 and data_proxy$index_B5
t = -1.8643, df = 136, p-value = 0.06444
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.316610047  0.009498434
sample estimates:
      cor 
-0.157857 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$oms_mal_co10 and data_proxy$index_B5
S = 470381, p-value = 0.3886
alternative hypothesis: true rho is not equal to 0
sample estimates:
        rho 
-0.07395657 </code></pre>
</div>
</div>
</section>
</div>
<blockquote class="blockquote">
<p><strong>The best proxies : A3 &gt; A1 &gt; B3 &gt; B1. From now we only focus on these indexes, and we just check on females as we have the same results.</strong></p>
</blockquote>
</section>
<section id="linear-models-and-r2" class="level3">
<h3 class="anchored" data-anchor-id="linear-models-and-r2">Linear models and R<sup>2</sup></h3>
<p><strong>A3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_fem_co10 ~ index_A3 + (1 | session)
   Data: data_proxy

REML criterion at convergence: 479

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-2.44166 -0.55337  0.08977  0.59748  2.08133 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 2.342    1.530   
 Residual             1.297    1.139   
Number of obs: 138, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept) 4.079e+00  4.405e-01 2.033e+01   9.260 9.79e-09 ***
index_A3    3.889e-02  4.743e-03 1.325e+02   8.201 1.82e-13 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
         (Intr)
index_A3 -0.441</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.732
     Marginal R2: 0.247</code></pre>
</div>
</div>
<p><strong>A1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_fem_co10 ~ index_A1 + (1 | session)
   Data: data_proxy

REML criterion at convergence: 476.1

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-2.43218 -0.52535  0.07917  0.55842  2.12807 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 2.963    1.721   
 Residual             1.278    1.131   
Number of obs: 138, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)   4.10334    0.48329  20.19537    8.49 4.26e-08 ***
index_A1      0.35020    0.04405 127.90824    7.95 8.48e-13 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
         (Intr)
index_A1 -0.407</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.754
     Marginal R2: 0.184</code></pre>
</div>
</div>
<p><strong>B3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_fem_co10 ~ index_B3 + (1 | session)
   Data: data_proxy

REML criterion at convergence: 517.2

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-2.29297 -0.58803  0.08513  0.61277  2.32355 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 3.955    1.989   
 Residual             1.613    1.270   
Number of obs: 138, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept) 5.038e+00  5.276e-01 1.664e+01   9.549 3.67e-08 ***
index_B3    3.053e-03  6.697e-04 1.270e+02   4.560 1.19e-05 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
         (Intr)
index_B3 -0.261</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.729
     Marginal R2: 0.065</code></pre>
</div>
</div>
<p><strong>B1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_fem_co10 ~ index_B1 + (1 | session)
   Data: data_proxy

REML criterion at convergence: 512.2

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-2.27002 -0.56989  0.09919  0.59569  2.33706 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 4.14     2.035   
 Residual             1.60     1.265   
Number of obs: 138, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept) 5.006e+00  5.399e-01 1.690e+01   9.272 4.88e-08 ***
index_B1    2.990e-02  6.508e-03 1.257e+02   4.594 1.04e-05 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
         (Intr)
index_B1 -0.266</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.738
     Marginal R2: 0.059</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Based on marginal, A3 is the best proxy, but based on conditionnal, A1 is better.</strong></p>
</blockquote>
</section>
<section id="aic" class="level3">
<h3 class="anchored" data-anchor-id="aic">AIC</h3>
<p><strong>A3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 486.9907</code></pre>
</div>
</div>
<p><strong>A1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 484.1203</code></pre>
</div>
</div>
<p><strong>B3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 525.1529</code></pre>
</div>
</div>
<p><strong>B1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 520.238</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Best AIC is for A1.</strong></p>
</blockquote>
</section>
<section id="root-mean-squared-error" class="level3">
<h3 class="anchored" data-anchor-id="root-mean-squared-error">Root Mean Squared Error</h3>
<p><strong>A3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] "RMSE: 1.07060689384694"</code></pre>
</div>
</div>
<p><strong>A1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] "RMSE: 1.06208041730774"</code></pre>
</div>
</div>
<p><strong>B3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] "RMSE: 1.19281847098643"</code></pre>
</div>
</div>
<p><strong>B1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] "RMSE: 1.18765895202105"</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Best based on RMSE is A1.</strong></p>
</blockquote>
</section>
<section id="conclusions" class="level3">
<h3 class="anchored" data-anchor-id="conclusions">Conclusions</h3>
<blockquote class="blockquote">
<p>To summarize, the number of contacts, and the number of contacts x max_oms, seems to be both good predictors of observational mating success. To do : Bateman gradients with the number of contacts to see if we obtain the same results? Necessary?</p>
</blockquote>
</section>
</section>
<section id="which-traits-are-good-predictor-of-mate-access" class="level2">
<h2 class="anchored" data-anchor-id="which-traits-are-good-predictor-of-mate-access">Which traits are good predictor of mate access?</h2>
<p>We measured:</p>
<ul>
<li><strong>nb_flo:</strong> the number of opened flowers</li>
<li><strong>nb_flo_open:</strong> the number of opened flowers (including very new ones)</li>
<li><strong>nb_flo_all:</strong> the total number of flowers (including very new and old ones)</li>
<li><strong>height_max:</strong> the height of the highest flower</li>
<li><strong>height_mean:</strong> the mean height of the highest flowers on each stem</li>
</ul>
<p>We analyse the correlation between these traits and the number of contacts, which had in turn an effect on oms as we saw above.</p>
<section id="simple-linear-models" class="level3">
<h3 class="anchored" data-anchor-id="simple-linear-models">Simple linear models</h3>
<section id="number-of-contacts" class="level4">
<h4 class="anchored" data-anchor-id="number-of-contacts">Number of contacts</h4>
<p>Simple linear models, regression of number of contacts on each trait.</p>
<p><strong>nb_flo</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: index_A1 ~ nb_flo + (1 | session)
   Data: data_proxy

REML criterion at convergence: 646.5

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.1467 -0.5929 -0.0645  0.5791  4.2087 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 3.221    1.795   
 Residual             5.035    2.244   
Number of obs: 138, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)   3.38887    0.72084  47.97546   4.701 2.21e-05 ***
nb_flo        0.09234    0.04392 135.66896   2.103   0.0373 *  
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
       (Intr)
nb_flo -0.735</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.409
     Marginal R2: 0.031</code></pre>
</div>
</div>
<p><strong>nb_flo_open</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: index_A1 ~ nb_flo_open + (1 | session)
   Data: data_proxy

REML criterion at convergence: 647.2

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.1711 -0.6088 -0.0663  0.6163  4.1507 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 3.198    1.788   
 Residual             5.053    2.248   
Number of obs: 138, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)   3.45356    0.71310  48.31683   4.843 1.36e-05 ***
nb_flo_open   0.07182    0.03561 135.99881   2.017   0.0457 *  
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
            (Intr)
nb_flo_open -0.730</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.404
     Marginal R2: 0.028</code></pre>
</div>
</div>
<p><strong>nb_flo_all</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: index_A1 ~ nb_flo_all + (1 | session)
   Data: data_proxy

REML criterion at convergence: 648.1

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.0755 -0.6367 -0.1438  0.5921  4.0907 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 3.272    1.809   
 Residual             5.045    2.246   
Number of obs: 138, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)   3.28498    0.78258  53.10105   4.198 0.000103 ***
nb_flo_all    0.04793    0.02396 132.55053   2.000 0.047513 *  
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
           (Intr)
nb_flo_all -0.777</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.412
     Marginal R2: 0.031</code></pre>
</div>
</div>
<p><strong>height_max</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: index_A1 ~ height_max + (1 | session)
   Data: data_proxy

REML criterion at convergence: 658

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.2822 -0.6331 -0.1178  0.6489  4.2839 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 2.843    1.686   
 Residual             5.219    2.285   
Number of obs: 138, groups:  session, 16

Fixed effects:
              Estimate Std. Error         df t value Pr(&gt;|t|)    
(Intercept)  4.560e+00  4.677e-01  1.476e+01   9.751 8.05e-08 ***
height_max  -7.015e-04  5.975e-04  1.295e+02  -1.174    0.243    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
           (Intr)
height_max -0.093</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.357
     Marginal R2: 0.007</code></pre>
</div>
</div>
<p><strong>height_mean</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: index_A1 ~ height_mean + (1 | session)
   Data: data_proxy

REML criterion at convergence: 655.3

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.2787 -0.6326 -0.1188  0.6486  4.2767 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 2.849    1.688   
 Residual             5.225    2.286   
Number of obs: 138, groups:  session, 16

Fixed effects:
              Estimate Std. Error         df t value Pr(&gt;|t|)    
(Intercept)   4.610338   0.475040  15.583025   9.705 5.27e-08 ***
height_mean  -0.002655   0.002404 129.745261  -1.104    0.272    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
            (Intr)
height_mean -0.193</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.357
     Marginal R2: 0.007</code></pre>
</div>
</div>
<p><strong>all traits together</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: 
index_A1 ~ nb_flo + nb_flo_open + nb_flo_all + height_max + height_mean +  
    (1 | session)
   Data: data_proxy

REML criterion at convergence: 665.3

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.2031 -0.6192 -0.0785  0.5417  4.3237 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 3.490    1.868   
 Residual             4.934    2.221   
Number of obs: 138, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)  
(Intercept)   1.04554    1.41110  94.66569   0.741   0.4606  
nb_flo        0.05477    0.12989 120.23844   0.422   0.6741  
nb_flo_open   0.01732    0.10584 120.37414   0.164   0.8703  
nb_flo_all    0.02136    0.03852 131.87672   0.555   0.5802  
height_max   -0.02627    0.01402 131.41703  -1.874   0.0631 .
height_mean   0.10256    0.05641 131.36911   1.818   0.0713 .
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
            (Intr) nb_flo nb_fl_p nb_fl_l hght_mx
nb_flo       0.023                               
nb_flo_open -0.111 -0.847                        
nb_flo_all  -0.164 -0.207 -0.226                 
height_max   0.826  0.070 -0.134   0.061         
height_mean -0.827 -0.073  0.137  -0.063  -0.999 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.460
     Marginal R2: 0.079</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>For simple linear models, only flower number have an effect on the number of contacts, and the strongest effect seems for the number of opened flowers (without very new or old ones). When working with all traits together, on the contrary, only height have an effect (negative for maximum, positive for mean)…</strong></p>
</blockquote>
</section>
<section id="observational-mating-success" class="level4">
<h4 class="anchored" data-anchor-id="observational-mating-success">Observational mating success</h4>
<p>Simple linear models, regression of observational mating success on each trait.</p>
<p><strong>nb_flo</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_fem_co10 ~ nb_flo + (1 | session)
   Data: data_proxy

REML criterion at convergence: 681.4

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-3.3061 -0.6045  0.1055  0.6192  1.9412 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 7.329    2.707   
 Residual             2.955    1.719   
Number of obs: 160, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)   4.78290    0.78828  24.71397   6.068 2.55e-06 ***
nb_flo        0.03364    0.03116 151.93297   1.080    0.282    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
       (Intr)
nb_flo -0.483</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.714
     Marginal R2: 0.004</code></pre>
</div>
</div>
<p><strong>nb_flo_open</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_fem_co10 ~ nb_flo_open + (1 | session)
   Data: data_proxy

REML criterion at convergence: 681.6

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-3.2961 -0.6044  0.1310  0.6117  1.9786 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 7.342    2.710   
 Residual             2.948    1.717   
Number of obs: 160, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)   4.75019    0.77947  23.70982   6.094 2.85e-06 ***
nb_flo_open   0.02997    0.02439 150.78777   1.229    0.221    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
            (Intr)
nb_flo_open -0.463</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.715
     Marginal R2: 0.005</code></pre>
</div>
</div>
<p><strong>nb_flo_all</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_fem_co10 ~ nb_flo_all + (1 | session)
   Data: data_proxy

REML criterion at convergence: 683.6

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-3.3397 -0.6240  0.0752  0.6436  1.9536 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 7.318    2.705   
 Residual             2.976    1.725   
Number of obs: 160, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)   5.00080    0.82895  29.36752   6.033 1.38e-06 ***
nb_flo_all    0.00745    0.01774 154.26072   0.420    0.675    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
           (Intr)
nb_flo_all -0.554</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.711
     Marginal R2: 0.001</code></pre>
</div>
</div>
<p><strong>height_max</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_fem_co10 ~ height_max + (1 | session)
   Data: data_proxy

REML criterion at convergence: 691.1

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-3.3447 -0.6319  0.1082  0.6507  1.9324 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 7.259    2.694   
 Residual             2.982    1.727   
Number of obs: 160, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept) 5.192e+00  6.877e-01 1.503e+01   7.549 1.72e-06 ***
height_max  2.936e-05  4.432e-04 1.442e+02   0.066    0.947    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
           (Intr)
height_max -0.037</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.709
     Marginal R2: 0.000</code></pre>
</div>
</div>
<p><strong>height_mean</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_fem_co10 ~ height_mean + (1 | session)
   Data: data_proxy

REML criterion at convergence: 688.3

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-3.3448 -0.6319  0.1079  0.6506  1.9323 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 7.261    2.695   
 Residual             2.982    1.727   
Number of obs: 160, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept) 5.188e+00  6.901e-01 1.523e+01   7.518 1.67e-06 ***
height_mean 1.799e-04  1.781e-03 1.443e+02   0.101     0.92    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
            (Intr)
height_mean -0.088</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.709
     Marginal R2: 0.000</code></pre>
</div>
</div>
<p><strong>all traits together</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_fem_co10 ~ nb_flo + nb_flo_open + nb_flo_all + height_max +  
    height_mean + (1 | session)
   Data: data_proxy

REML criterion at convergence: 705.7

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-3.2218 -0.6044  0.1581  0.5915  1.8942 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 7.178    2.679   
 Residual             2.993    1.730   
Number of obs: 160, groups:  session, 16

Fixed effects:
              Estimate Std. Error         df t value Pr(&gt;|t|)   
(Intercept)  3.810e+00  1.248e+00  8.899e+01   3.052  0.00299 **
nb_flo      -5.824e-04  9.234e-02  1.408e+02  -0.006  0.99498   
nb_flo_open  6.443e-02  7.044e-02  1.398e+02   0.915  0.36195   
nb_flo_all  -2.439e-02  2.896e-02  1.481e+02  -0.842  0.40103   
height_max  -1.333e-02  1.080e-02  1.498e+02  -1.234  0.21901   
height_mean  5.385e-02  4.343e-02  1.498e+02   1.240  0.21696   
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
            (Intr) nb_flo nb_fl_p nb_fl_l hght_mx
nb_flo       0.011                               
nb_flo_open -0.112 -0.832                        
nb_flo_all  -0.165 -0.304 -0.146                 
height_max   0.749  0.046 -0.164   0.057         
height_mean -0.750 -0.050  0.168  -0.057  -0.999 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.710
     Marginal R2: 0.016</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Concerning the regression of the observational mating success on phenotypic trait, we do not detected any effect (both for simple linear models or when working with all traits together).</strong></p>
</blockquote>
</section>
</section>
<section id="sem" class="level3">
<h3 class="anchored" data-anchor-id="sem">SEM</h3>
<p>Best way to do this analysis: SEM?</p>
<div class="cell">
<div class="cell-output-display">
<div class="grViz html-widget html-fill-item" id="htmlwidget-058458f65cdbb4d74ae0" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-058458f65cdbb4d74ae0">{"x":{"diagram":"digraph {\n\ngraph [layout = \"neato\",\n       outputorder = \"edgesfirst\",\n       bgcolor = \"white\"]\n\nnode [fontname = \"Helvetica\",\n      fontsize = \"10\",\n      shape = \"circle\",\n      fixedsize = \"true\",\n      width = \"0.5\",\n      style = \"filled\",\n      fillcolor = \"aliceblue\",\n      color = \"gray70\",\n      fontcolor = \"gray50\"]\n\nedge [fontname = \"Helvetica\",\n     fontsize = \"8\",\n     len = \"1.5\",\n     color = \"gray80\",\n     arrowsize = \"0.5\"]\n\n  \"1\" [label = \"index_A1\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2.5,2!\"] \n  \"2\" [label = \"oms_fem_co10\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2.5,3!\"] \n  \"3\" [label = \"nb_flo\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"1,1!\"] \n  \"4\" [label = \"nb_flo_open\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,1!\"] \n  \"5\" [label = \"nb_flo_all\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"3,1!\"] \n  \"6\" [label = \"height_max\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"4,1!\"] \n  \"7\" [label = \"height_mean\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"5,1!\"] \n\"3\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.118 \"] \n\"4\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.046 \"] \n\"5\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.088 \"] \n\"6\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" -3.058 \"] \n\"7\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 2.978 \"] \n\"1\"->\"2\" [fontsize = \"6\", fontsize = \"6\", label = \" 0.315 \"] \n\"3\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" -0.071 \"] \n\"4\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.135 \"] \n\"5\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" -0.036 \"] \n\"6\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 1.546 \"] \n\"7\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" -1.562 \"] \n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Structural Equation Model of psem_proxy 

Call:
  index_A1 ~ nb_flo + nb_flo_open + nb_flo_all + height_max + height_mean
  oms_fem_co10 ~ index_A1 + nb_flo + nb_flo_open + nb_flo_all + height_max + height_mean

    AIC
 1203.577

---
Tests of directed separation:

 tableau de données (data frame) avec 0 colonnes et 0 ligne

--
Global goodness-of-fit:

Chi-Squared = 0 with P-value = 1 and on 0 degrees of freedom
Fisher's C = NA with P-value = NA and on 0 degrees of freedom

---
Coefficients:

      Response   Predictor Estimate Std.Error       DF Crit.Value P.Value
      index_A1      nb_flo   0.0548    0.1299 120.4409     0.1771  0.6747
      index_A1 nb_flo_open   0.0173    0.1058 120.5745     0.0267  0.8706
      index_A1  nb_flo_all   0.0214    0.0385 131.8790     0.2977  0.5863
      index_A1  height_max  -0.0263    0.0140 131.4280     3.4056  0.0672
      index_A1 height_mean   0.1026    0.0564 131.3810     3.2052  0.0757
  oms_fem_co10    index_A1   0.3541    0.0456 124.2425    59.6741  0.0000
  oms_fem_co10      nb_flo  -0.0369    0.0658 117.3790     0.3143  0.5761
  oms_fem_co10 nb_flo_open   0.0570    0.0535 117.2525     1.1334  0.2892
  oms_fem_co10  nb_flo_all  -0.0098    0.0203 124.7255     0.2303  0.6321
  oms_fem_co10  height_max   0.0149    0.0075 127.0914     3.8426  0.0522
  oms_fem_co10 height_mean  -0.0604    0.0303 127.1391     3.8980  0.0505
  Std.Estimate    
        0.1183    
        0.0460    
        0.0877    
       -3.0578    
        2.9776    
        0.3154 ***
       -0.0710    
        0.1351    
       -0.0358    
        1.5455    
       -1.5616    

  Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05

---
Individual R-squared:

      Response method Marginal Conditional
      index_A1   none     0.08        0.46
  oms_fem_co10   none     0.19        0.77</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Strongest effect for the height, with a negative effect of maximum height and a positive effect for mean height on the number of contacts, and the opposite effects for the observational mating success</strong>.</p>
</blockquote>
</section>
</section>
</section>
<section id="comparing-observational-to-genetic-mating-success" class="level1">
<h1>Comparing observational to genetic mating success</h1>
<section id="true-batemans-gradient-on-all-data-based-on-observations" class="level2">
<h2 class="anchored" data-anchor-id="true-batemans-gradient-on-all-data-based-on-observations">True Bateman’s gradient on all data based on observations</h2>
<p>We stay consistent with the manuscript and work with outcrossed reproductive success, estimated on all genotypes, and the ratio is then multiplied by the number of total seeds produced per female, both for the female and the male function (i.e.&nbsp;paternity share).</p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-58-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="bateman-hedges" class="level2">
<h2 class="anchored" data-anchor-id="bateman-hedges">Bateman Hedge’s</h2>
<p>Difference between Bateman’s estimates with genetic sampling versus true Bateman with observational data (sampling gms - true oms). An Hedges’ g &gt; 0 mean that the Bateman gradient estimated with sampling gms is greater than the real ones with true oms.</p>
<p>Observational versus genetic, according to the sampling percentage of genotyped seeds.</p>
<section id="per-session" class="level3">
<h3 class="anchored" data-anchor-id="per-session">Per session:</h3>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-59-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="overall" class="level3">
<h3 class="anchored" data-anchor-id="overall">Overall:</h3>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-60-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="variance-decomposition" class="level2">
<h2 class="anchored" data-anchor-id="variance-decomposition">Variance decomposition</h2>
<p>Observational (with and without flower) versus genetic, according to the sampling percentage of genotyped seeds</p>

</section>
</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body" role="list">
<div id="ref-cresswell_method_1994" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Cresswell JE. 1994 A method for quantifying the gene flow that results from a single bumblebee visit using transgenic oilseed rape,<em><span>Brassica</span> napus</em> <span>L</span>. Cv. <span>Westar</span>. <em>Transgenic Research</em> <strong>3</strong>, 134–137. (doi:<a href="https://doi.org/10.1007/BF01974092">10.1007/BF01974092</a>)</div>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>