<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.36">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Sampling oms</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting-01c78b5cd655e4cd89133cf59d535862.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap-11df3c80491ea81009d83f04e2a62a23.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<link href="index_files/libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="index_files/libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="index_files/libs/vis-9.1.0/vis-network.min.css" rel="stylesheet">
<script src="index_files/libs/vis-9.1.0/vis-network.min.js"></script>
<script src="index_files/libs/visNetwork-binding-2.1.2/visNetwork.js"></script>
<script src="index_files/libs/viz-1.8.2/viz.js"></script>
<link href="index_files/libs/DiagrammeR-styles-0.2/styles.css" rel="stylesheet">
<script src="index_files/libs/grViz-binding-1.0.11/grViz.js"></script>


</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#project-and-data-vizualisation" id="toc-project-and-data-vizualisation" class="nav-link active" data-scroll-target="#project-and-data-vizualisation">Project and data vizualisation</a></li>
  <li><a href="#what-is-the-best-proxy-for-observational-mating-success" id="toc-what-is-the-best-proxy-for-observational-mating-success" class="nav-link" data-scroll-target="#what-is-the-best-proxy-for-observational-mating-success">What is the best proxy for observational mating success?</a>
  <ul class="collapse">
  <li><a href="#visit-sequences" id="toc-visit-sequences" class="nav-link" data-scroll-target="#visit-sequences">Visit sequences</a>
  <ul class="collapse">
  <li><a href="#acp" id="toc-acp" class="nav-link" data-scroll-target="#acp">ACP</a></li>
  <li><a href="#correlation-test" id="toc-correlation-test" class="nav-link" data-scroll-target="#correlation-test">Correlation test</a></li>
  <li><a href="#linear-models-and-r2" id="toc-linear-models-and-r2" class="nav-link" data-scroll-target="#linear-models-and-r2">Linear models and R<sup>2</sup></a></li>
  <li><a href="#aic" id="toc-aic" class="nav-link" data-scroll-target="#aic">AIC</a></li>
  <li><a href="#root-mean-squared-error" id="toc-root-mean-squared-error" class="nav-link" data-scroll-target="#root-mean-squared-error">Root Mean Squared Error</a></li>
  <li><a href="#conclusions" id="toc-conclusions" class="nav-link" data-scroll-target="#conclusions">Conclusions</a></li>
  <li><a href="#plots" id="toc-plots" class="nav-link" data-scroll-target="#plots">Plots</a></li>
  </ul></li>
  <li><a href="#which-traits-are-good-predictor-of-mate-access" id="toc-which-traits-are-good-predictor-of-mate-access" class="nav-link" data-scroll-target="#which-traits-are-good-predictor-of-mate-access">Which traits are good predictor of mate access?</a>
  <ul class="collapse">
  <li><a href="#simple-linear-models" id="toc-simple-linear-models" class="nav-link" data-scroll-target="#simple-linear-models">Simple linear models</a></li>
  <li><a href="#sem" id="toc-sem" class="nav-link" data-scroll-target="#sem">SEM</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#comparing-observational-to-genetic-mating-success" id="toc-comparing-observational-to-genetic-mating-success" class="nav-link" data-scroll-target="#comparing-observational-to-genetic-mating-success">Comparing observational to genetic mating success</a>
  <ul class="collapse">
  <li><a href="#true-batemans-gradient-on-all-data-based-on-observations" id="toc-true-batemans-gradient-on-all-data-based-on-observations" class="nav-link" data-scroll-target="#true-batemans-gradient-on-all-data-based-on-observations">True Bateman’s gradient on all data based on observations</a></li>
  <li><a href="#bateman-hedges" id="toc-bateman-hedges" class="nav-link" data-scroll-target="#bateman-hedges">Bateman Hedge’s</a>
  <ul class="collapse">
  <li><a href="#per-session" id="toc-per-session" class="nav-link" data-scroll-target="#per-session">Per session:</a></li>
  <li><a href="#overall" id="toc-overall" class="nav-link" data-scroll-target="#overall">Overall:</a></li>
  </ul></li>
  <li><a href="#variance-decomposition" id="toc-variance-decomposition" class="nav-link" data-scroll-target="#variance-decomposition">Variance decomposition</a></li>
  </ul></li>
  </ul>
</nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Sampling oms</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="project-and-data-vizualisation" class="level1">
<h1>Project and data vizualisation</h1>
<p>Visualize the pipeline analyses:</p>
<div class="cell">
<div class="cell-output-display">
<div class="visNetwork html-widget html-fill-item" id="htmlwidget-d84d8c298f805d8130f5" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-d84d8c298f805d8130f5">{"x":{"nodes":{"name":["data_contact","data_genotypes","data_hedges_false","data_hedges_true","data_id","data_id_sampled_sessions","data_obs","data_proxy","data_sampling","data_sampling_bateman","data_sampling_rs_ms","data_sem_complete_sessions","data_sem_sampled_sessions","data_session","data_true_bateman","data_true_rs_ms","data_var_gms","index","linear_models_oms_proxy","pcas_10","pcas_20","pcas_5","predictions_oms_proxy","get_data_contact","load_data_id","get_sampling_bateman","get_data_sem_sampled_sessions","compute_predictions_oms_proxy","load_data_obs","get_data_session","get_sampling_rs_ms","get_true_rs_ms","load_data","get_plot_gms","get_linear_models_oms_proxy","get_hedges","get_pca","bootstrap_mean_ci","compute_oms_and_proxy","get_var_gms","get_data_sem_complete_sessions","get_true_bateman"],"type":["stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","stem","function","function","function","function","function","function","function","function","function","function","function","function","function","function","function","function","function","function","function"],"description":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"status":["uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","dispatched","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate","uptodate"],"seconds":[0.946,0.003,0.029,0.11,0.059,0.052,0.011,0.052,12.466,687.8630000000001,1.626,0.019,0.038,0.037,1.417,0.106,148.443,14.511,0.928,0.54,0.581,1.055,0.749,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"bytes":[18013,13324,531220,573957,3250,518458,49381,27775,63177025,7627687,9587597,2026,5309,558,2605,2195,42879,6794677,565547,566852,561175,569936,16901,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"branches":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"label":["data_contact","data_genotypes","data_hedges_false","data_hedges_true","data_id","data_id_sampled_sessions","data_obs","data_proxy","data_sampling","data_sampling_bateman","data_sampling_rs_ms","data_sem_complete_sessions","data_sem_sampled_sessions","data_session","data_true_bateman","data_true_rs_ms","data_var_gms","index","linear_models_oms_proxy","pcas_10","pcas_20","pcas_5","predictions_oms_proxy","get_data_contact","load_data_id","get_sampling_bateman","get_data_sem_sampled_sessions","compute_predictions_oms_proxy","load_data_obs","get_data_session","get_sampling_rs_ms","get_true_rs_ms","load_data","get_plot_gms","get_linear_models_oms_proxy","get_hedges","get_pca","bootstrap_mean_ci","compute_oms_and_proxy","get_var_gms","get_data_sem_complete_sessions","get_true_bateman"],"color":["#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#DC863B","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823","#354823"],"id":["data_contact","data_genotypes","data_hedges_false","data_hedges_true","data_id","data_id_sampled_sessions","data_obs","data_proxy","data_sampling","data_sampling_bateman","data_sampling_rs_ms","data_sem_complete_sessions","data_sem_sampled_sessions","data_session","data_true_bateman","data_true_rs_ms","data_var_gms","index","linear_models_oms_proxy","pcas_10","pcas_20","pcas_5","predictions_oms_proxy","get_data_contact","load_data_id","get_sampling_bateman","get_data_sem_sampled_sessions","compute_predictions_oms_proxy","load_data_obs","get_data_session","get_sampling_rs_ms","get_true_rs_ms","load_data","get_plot_gms","get_linear_models_oms_proxy","get_hedges","get_pca","bootstrap_mean_ci","compute_oms_and_proxy","get_var_gms","get_data_sem_complete_sessions","get_true_bateman"],"level":[3,2,7,7,2,2,2,5,2,3,2,6,6,4,6,3,3,8,6,6,6,6,7,1,1,1,1,1,1,1,0,1,1,0,1,1,1,0,1,1,1,1],"shape":["dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle","triangle"]},"edges":{"from":["data_proxy","get_pca","data_sampling_bateman","data_true_bateman","get_hedges","data_proxy","get_pca","data_contact","get_data_session","load_data","load_data_obs","data_id_sampled_sessions","data_proxy","get_data_sem_sampled_sessions","data_id","data_sampling","get_var_gms","data_sampling_bateman","data_true_bateman","get_hedges","data_sampling_rs_ms","get_sampling_bateman","compute_predictions_oms_proxy","data_proxy","linear_models_oms_proxy","load_data","data_proxy","data_true_rs_ms","get_true_bateman","load_data","data_genotypes","data_id","get_true_rs_ms","load_data","data_proxy","get_pca","data_obs","get_data_contact","data_contact","data_hedges_false","data_hedges_true","data_id","data_obs","data_proxy","data_sem_complete_sessions","data_sem_sampled_sessions","data_session","data_true_bateman","linear_models_oms_proxy","pcas_10","pcas_20","pcas_5","predictions_oms_proxy","data_proxy","data_true_rs_ms","get_data_sem_complete_sessions","load_data_id","compute_oms_and_proxy","data_contact","data_id","data_session","data_proxy","get_linear_models_oms_proxy"],"to":["pcas_5","pcas_5","data_hedges_true","data_hedges_true","data_hedges_true","pcas_20","pcas_20","data_session","data_session","data_sampling","data_obs","data_sem_sampled_sessions","data_sem_sampled_sessions","data_sem_sampled_sessions","data_var_gms","data_var_gms","data_var_gms","data_hedges_false","data_hedges_false","data_hedges_false","data_sampling_bateman","data_sampling_bateman","predictions_oms_proxy","predictions_oms_proxy","predictions_oms_proxy","data_sampling_rs_ms","data_true_bateman","data_true_bateman","data_true_bateman","data_genotypes","data_true_rs_ms","data_true_rs_ms","data_true_rs_ms","data_id_sampled_sessions","pcas_10","pcas_10","data_contact","data_contact","index","index","index","index","index","index","index","index","index","index","index","index","index","index","index","data_sem_complete_sessions","data_sem_complete_sessions","data_sem_complete_sessions","data_id","data_proxy","data_proxy","data_proxy","data_proxy","linear_models_oms_proxy","linear_models_oms_proxy"],"arrows":["to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot","physics":false},"manipulation":{"enabled":false},"edges":{"smooth":{"type":"cubicBezier","forceDirection":"horizontal"}},"physics":{"stabilization":false},"interaction":{"zoomSpeed":1},"layout":{"hierarchical":{"enabled":true,"direction":"LR"}}},"groups":null,"width":null,"height":null,"idselection":{"enabled":false,"style":"width: 150px; height: 26px","useLabels":true,"main":"Select by id"},"byselection":{"enabled":false,"style":"width: 150px; height: 26px","multiple":false,"hideColor":"rgba(200,200,200,0.5)","highlight":false},"main":{"text":"","style":"font-family:Georgia, Times New Roman, Times, serif;font-weight:bold;font-size:20px;text-align:center;"},"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","highlight":{"enabled":true,"hoverNearest":false,"degree":{"from":1,"to":1},"algorithm":"hierarchical","hideColor":"rgba(200,200,200,0.5)","labelOnly":true},"collapse":{"enabled":true,"fit":false,"resetHighlight":true,"clusterOptions":null,"keepCoord":true,"labelSuffix":"(cluster)"},"legend":{"width":0.2,"useGroups":false,"position":"right","ncol":1,"stepX":100,"stepY":100,"zoom":true,"nodes":{"label":["Up to date","Dispatched","Stem","Function"],"color":["#354823","#DC863B","#899DA4","#899DA4"],"shape":["dot","dot","dot","triangle"]},"nodesToDataframe":true},"tooltipStay":300,"tooltipStyle":"position: fixed;visibility:hidden;padding: 5px;white-space: nowrap;font-family: verdana;font-size:14px;font-color:#000000;background-color: #f5f4ed;-moz-border-radius: 3px;-webkit-border-radius: 3px;border-radius: 3px;border: 1px solid #808074;box-shadow: 3px 3px 10px rgba(0, 0, 0, 0.2);"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 4%">
<col style="width: 5%">
<col style="width: 7%">
<col style="width: 4%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 8%">
<col style="width: 6%">
<col style="width: 7%">
<col style="width: 7%">
<col style="width: 6%">
<col style="width: 7%">
<col style="width: 7%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">session</th>
<th style="text-align: left;">id</th>
<th style="text-align: right;">sr_fem_total</th>
<th style="text-align: right;">nb_flo</th>
<th style="text-align: right;">nb_flo_open</th>
<th style="text-align: right;">nb_flo_all</th>
<th style="text-align: right;">height_max</th>
<th style="text-align: right;">height_mean</th>
<th style="text-align: right;">nb_poll_focal</th>
<th style="text-align: right;">oms_fem_co5</th>
<th style="text-align: right;">oms_fem_co10</th>
<th style="text-align: right;">oms_fem_co20</th>
<th style="text-align: right;">oms_mal_co5</th>
<th style="text-align: right;">oms_mal_co10</th>
<th style="text-align: right;">oms_mal_co20</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">56</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">20</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">40.4</td>
<td style="text-align: right;">36.73333</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.02</td>
<td style="text-align: right;">119</td>
<td style="text-align: right;">19</td>
<td style="text-align: right;">21</td>
<td style="text-align: right;">26</td>
<td style="text-align: right;">31.5</td>
<td style="text-align: right;">24.40000</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.03</td>
<td style="text-align: right;">38</td>
<td style="text-align: right;">11</td>
<td style="text-align: right;">13</td>
<td style="text-align: right;">13</td>
<td style="text-align: right;">37.4</td>
<td style="text-align: right;">35.45000</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.04</td>
<td style="text-align: right;">169</td>
<td style="text-align: right;">19</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">28</td>
<td style="text-align: right;">36.4</td>
<td style="text-align: right;">33.33333</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.05</td>
<td style="text-align: right;">29</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">18</td>
<td style="text-align: right;">36.5</td>
<td style="text-align: right;">31.63333</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.06</td>
<td style="text-align: right;">130</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">22</td>
<td style="text-align: right;">30</td>
<td style="text-align: right;">40.3</td>
<td style="text-align: right;">34.35000</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
</tr>
</tbody>
</table>
</div>
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 9%">
<col style="width: 8%">
<col style="width: 8%">
<col style="width: 11%">
<col style="width: 11%">
<col style="width: 3%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 9%">
<col style="width: 17%">
<col style="width: 7%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">session</th>
<th style="text-align: left;">people</th>
<th style="text-align: right;">time</th>
<th style="text-align: right;">duration</th>
<th style="text-align: left;">ID_full</th>
<th style="text-align: right;">ID</th>
<th style="text-align: right;">noHp</th>
<th style="text-align: left;">flow</th>
<th style="text-align: left;">id_flow</th>
<th style="text-align: left;">id_flow_full</th>
<th style="text-align: left;">ttt</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: right;">32.768</td>
<td style="text-align: right;">2.815</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">3</td>
<td style="text-align: left;">1_1_3</td>
<td style="text-align: left;">1.FA1.01_1_3</td>
<td style="text-align: left;">1_low</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: right;">36.822</td>
<td style="text-align: right;">3.327</td>
<td style="text-align: left;">1.FA1.06</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">2</td>
<td style="text-align: left;">NR</td>
<td style="text-align: left;">6_2_NR</td>
<td style="text-align: left;">1.FA1.06_2_NR</td>
<td style="text-align: left;">1_low</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: right;">56.448</td>
<td style="text-align: right;">8.191</td>
<td style="text-align: left;">1.FA1.03</td>
<td style="text-align: right;">3</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">8</td>
<td style="text-align: left;">3_1_8</td>
<td style="text-align: left;">1.FA1.03_1_8</td>
<td style="text-align: left;">1_low</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: right;">67.371</td>
<td style="text-align: right;">12.628</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">NR</td>
<td style="text-align: left;">1_1_NR</td>
<td style="text-align: left;">1.FA1.01_1_NR</td>
<td style="text-align: left;">1_low</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: right;">80.000</td>
<td style="text-align: right;">1.023</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">3</td>
<td style="text-align: left;">1_1_3</td>
<td style="text-align: left;">1.FA1.01_1_3</td>
<td style="text-align: left;">1_low</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: right;">81.024</td>
<td style="text-align: right;">28.650</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: left;">4</td>
<td style="text-align: left;">1_1_4</td>
<td style="text-align: left;">1.FA1.01_1_4</td>
<td style="text-align: left;">1_low</td>
</tr>
</tbody>
</table>
</div>
</div>
</section>
<section id="what-is-the-best-proxy-for-observational-mating-success" class="level1">
<h1>What is the best proxy for observational mating success?</h1>
<section id="visit-sequences" class="level2">
<h2 class="anchored" data-anchor-id="visit-sequences">Visit sequences</h2>
<p>We have conducted an experiment in which we surveyed each pollinator one by one along the visit sequences, up to the flower scale. By making assumptions about the carry-over (i.e., the number of successive flower visits during which pollen export remains effective), we are able to estimate the number of observational sexual partners, both for the female and the male function. We assume that the carry-over is about 10, based on a previous study on the sister species <em>Brassica napus</em>: for the male function, 91% of the seeds were sired in the first four flowers visited after a focal individual, and no seeds were sired after the 14th flower visited <span class="citation" data-cites="cresswell_method_1994">[<a href="#ref-cresswell_method_1994" role="doc-biblioref">1</a>]</span>.</p>
<p>Now, we imagine a more common experiment <em>in natura</em> in which we cannot follow the pollinators one by one. We could rather observe plant by plant. We then have the following data :</p>
<ul>
<li>the number of arrivals (female function) / departures (male function) per individual, which are highly correlated = number of contacts (which may be independent or not) per individual<br>
</li>
<li>the total number of visits per individual<br>
</li>
<li>the total visit duration per individual (but not useful here, as we only formulate hypotheses about visit sequences, without getting interested in the number of pollen grains exported/deposited): we leave that aside</li>
</ul>
<p>We can imagine that each observer notes the time of the pollinator’s arrival on each plant (start), to order events of contacts:</p>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 8%">
<col style="width: 7%">
<col style="width: 10%">
<col style="width: 22%">
<col style="width: 8%">
<col style="width: 10%">
<col style="width: 10%">
<col style="width: 21%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">session</th>
<th style="text-align: left;">people</th>
<th style="text-align: left;">id</th>
<th style="text-align: right;">consecutive_contact</th>
<th style="text-align: right;">start</th>
<th style="text-align: right;">nb_visit</th>
<th style="text-align: right;">duration</th>
<th style="text-align: right;">no_contact_with_id</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">32.768</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">2.815</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: left;">1.FA1.06</td>
<td style="text-align: right;">2</td>
<td style="text-align: right;">36.822</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">3.327</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: left;">1.FA1.03</td>
<td style="text-align: right;">3</td>
<td style="text-align: right;">56.448</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8.191</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">4</td>
<td style="text-align: right;">67.371</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">105.677</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: left;">1.FA1.06</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">183.382</td>
<td style="text-align: right;">2</td>
<td style="text-align: right;">10.877</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">Jeanne</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">201.195</td>
<td style="text-align: right;">2</td>
<td style="text-align: right;">13.056</td>
<td style="text-align: right;">3</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>But then, what to do with this? We don’t know the identity of the pollinator,<br>
so it is difficult to infer anything from the visit order…</p>
<p>Something more realistic is to start from data at the individual level without inferring anything about the effect of visit order:</p>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;">session</th>
<th style="text-align: left;">id</th>
<th style="text-align: right;">nb_contact</th>
<th style="text-align: right;">nb_visit</th>
<th style="text-align: right;">dur_visit</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">25</td>
<td style="text-align: right;">213.846</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.02</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">76.402</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.03</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">17</td>
<td style="text-align: right;">84.098</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.04</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">118.506</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.05</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">65.717</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.06</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">29</td>
<td style="text-align: right;">157.325</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>To discuss: something smart to do with contact order?</p>
<p>Then, we can summarize data at the scale of the observation session. At least useful to compare<br>
populations between them? We can get the following data:</p>
<ul>
<li>Total number of contacts with id</li>
<li>Total number of visits<br>
</li>
<li>Total number of visited individuals (maximum oms)<br>
</li>
<li>Total number of pollinators (but see below)<br>
</li>
<li>Total visit duration (but again, we leave that aside)</li>
</ul>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 10%">
<col style="width: 10%">
<col style="width: 22%">
<col style="width: 19%">
<col style="width: 19%">
<col style="width: 18%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">session</th>
<th style="text-align: right;">max_oms</th>
<th style="text-align: right;">total_nb_contact</th>
<th style="text-align: right;">total_nb_visit</th>
<th style="text-align: right;">total_duration</th>
<th style="text-align: right;">total_nb_poll</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA2</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">27</td>
<td style="text-align: right;">124</td>
<td style="text-align: right;">1098.092</td>
<td style="text-align: right;">1</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FO1</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">62</td>
<td style="text-align: right;">486</td>
<td style="text-align: right;">3642.479</td>
<td style="text-align: right;">4</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FO2</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">42</td>
<td style="text-align: right;">373</td>
<td style="text-align: right;">3178.549</td>
<td style="text-align: right;">3</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.MO1</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">50</td>
<td style="text-align: right;">185</td>
<td style="text-align: right;">1509.840</td>
<td style="text-align: right;">2</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.MO2</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">11</td>
<td style="text-align: right;">109</td>
<td style="text-align: right;">1463.275</td>
<td style="text-align: right;">2</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Because, imagine that one individual had a lot of contacts with pollinators, but<br>
that only a few other individuals were visited: this should weaken the link between<br>
the number of contacts per id and oms.</p>
<p>Using the data table with contacts ordered by time, we could imagine estimating<br>
the diversity of potential sexual partners according to their relative frequency<br>
(e.g., using Hill/Shannon/Simpson indices).</p>
<p>NB: In our experiment, we know for sure the number of pollinators that visited<br>
plants. <em>In natura</em>, it would be much more difficult, but we could at least get an estimation<br>
based on the species diversity that we observed (i.e., if we observed<br>
two contacts by <em>Bombus</em> and one contact by <em>Apis</em>, there are at least two different<br>
pollinators).</p>
<p>We now investigate which proxy best correlates with the observational number of mates.<br>
We estimate the latter using a carry-over of 10, as well as 5 and 20. We thus<br>
have two main proxies:</p>
<ul>
<li><ol type="A">
<li>the number of contacts per individual<br>
</li>
</ol></li>
<li><ol start="2" type="A">
<li>the total number of visits per individual</li>
</ol></li>
</ul>
<ol type="A">
<li>For the number of contacts per individual, we can explore its correlation with<br>
oms using:<br>
</li>
</ol>
<ul>
<li>index_A1: the raw number of contacts per individual<br>
</li>
<li>index_A2: relativized by the total number of contacts in the session<br>
</li>
<li>index_A3: multiplied by the total number of potential sexual partners<br>
</li>
<li>index_A4: combining index_A2 and index_A3<br>
</li>
</ul>
<ol start="2" type="A">
<li>For the total number of visits per individual:<br>
</li>
</ol>
<ul>
<li>index_B1: the raw number of visits per individual<br>
</li>
<li>index_B2: relativized by the total number of visits in the session<br>
</li>
<li>index_B3: multiplied by the total number of potential sexual partners<br>
</li>
<li>index_B4: combining index_B2 and index_B3<br>
</li>
</ul>
<p>After discussion with Jeanne, we also explore the same proxies based on total visit duration per individual:<br>
</p>
<ol start="3" type="A">
<li>For the total visit duration per individual:<br>
</li>
</ol>
<ul>
<li>index_C1: the raw total visit duration per individual<br>
</li>
<li>index_C2: relativized by the total visit duration in the session<br>
</li>
<li>index_C3: multiplied by the total number of potential sexual partners<br>
</li>
<li>index_C4: combining index_C2 and index_C3<br>
</li>
</ul>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 1%">
<col style="width: 2%">
<col style="width: 3%">
<col style="width: 1%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 3%">
<col style="width: 2%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 2%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 1%">
<col style="width: 4%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 3%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
<col style="width: 2%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">session</th>
<th style="text-align: left;">id</th>
<th style="text-align: right;">sr_fem_total</th>
<th style="text-align: right;">nb_flo</th>
<th style="text-align: right;">nb_flo_open</th>
<th style="text-align: right;">nb_flo_all</th>
<th style="text-align: right;">height_max</th>
<th style="text-align: right;">height_mean</th>
<th style="text-align: right;">nb_poll_focal</th>
<th style="text-align: right;">oms_fem_co5</th>
<th style="text-align: right;">oms_fem_co10</th>
<th style="text-align: right;">oms_fem_co20</th>
<th style="text-align: right;">oms_mal_co5</th>
<th style="text-align: right;">oms_mal_co10</th>
<th style="text-align: right;">oms_mal_co20</th>
<th style="text-align: right;">nb_contact</th>
<th style="text-align: right;">nb_visit</th>
<th style="text-align: right;">dur_visit</th>
<th style="text-align: right;">max_oms</th>
<th style="text-align: right;">total_nb_contact</th>
<th style="text-align: right;">total_nb_visit</th>
<th style="text-align: right;">total_duration</th>
<th style="text-align: right;">total_nb_poll</th>
<th style="text-align: right;">index_A1</th>
<th style="text-align: right;">index_A2</th>
<th style="text-align: right;">index_A3</th>
<th style="text-align: right;">index_A4</th>
<th style="text-align: right;">index_A5</th>
<th style="text-align: right;">index_B1</th>
<th style="text-align: right;">index_B2</th>
<th style="text-align: right;">index_B3</th>
<th style="text-align: right;">index_B4</th>
<th style="text-align: right;">index_B5</th>
<th style="text-align: right;">index_C1</th>
<th style="text-align: right;">index_C2</th>
<th style="text-align: right;">index_C3</th>
<th style="text-align: right;">index_C4</th>
<th style="text-align: right;">index_C5</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.01</td>
<td style="text-align: right;">56</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">20</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">40.4</td>
<td style="text-align: right;">36.73333</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">25</td>
<td style="text-align: right;">213.846</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">0.1369863</td>
<td style="text-align: right;">100</td>
<td style="text-align: right;">1.3698630</td>
<td style="text-align: right;">1.3698630</td>
<td style="text-align: right;">25</td>
<td style="text-align: right;">0.1269036</td>
<td style="text-align: right;">250</td>
<td style="text-align: right;">1.2690355</td>
<td style="text-align: right;">1.2690355</td>
<td style="text-align: right;">213.846</td>
<td style="text-align: right;">0.1786873</td>
<td style="text-align: right;">2138.46</td>
<td style="text-align: right;">1.7868731</td>
<td style="text-align: right;">1.7868731</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.02</td>
<td style="text-align: right;">119</td>
<td style="text-align: right;">19</td>
<td style="text-align: right;">21</td>
<td style="text-align: right;">26</td>
<td style="text-align: right;">31.5</td>
<td style="text-align: right;">24.40000</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">76.402</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">0.0684932</td>
<td style="text-align: right;">50</td>
<td style="text-align: right;">0.6849315</td>
<td style="text-align: right;">0.6849315</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">0.0710660</td>
<td style="text-align: right;">140</td>
<td style="text-align: right;">0.7106599</td>
<td style="text-align: right;">0.7106599</td>
<td style="text-align: right;">76.402</td>
<td style="text-align: right;">0.0638406</td>
<td style="text-align: right;">764.02</td>
<td style="text-align: right;">0.6384065</td>
<td style="text-align: right;">0.6384065</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.03</td>
<td style="text-align: right;">38</td>
<td style="text-align: right;">11</td>
<td style="text-align: right;">13</td>
<td style="text-align: right;">13</td>
<td style="text-align: right;">37.4</td>
<td style="text-align: right;">35.45000</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">17</td>
<td style="text-align: right;">84.098</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">0.1095890</td>
<td style="text-align: right;">80</td>
<td style="text-align: right;">1.0958904</td>
<td style="text-align: right;">1.0958904</td>
<td style="text-align: right;">17</td>
<td style="text-align: right;">0.0862944</td>
<td style="text-align: right;">170</td>
<td style="text-align: right;">0.8629442</td>
<td style="text-align: right;">0.8629442</td>
<td style="text-align: right;">84.098</td>
<td style="text-align: right;">0.0702713</td>
<td style="text-align: right;">840.98</td>
<td style="text-align: right;">0.7027134</td>
<td style="text-align: right;">0.7027134</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.04</td>
<td style="text-align: right;">169</td>
<td style="text-align: right;">19</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">28</td>
<td style="text-align: right;">36.4</td>
<td style="text-align: right;">33.33333</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">118.506</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">0.1095890</td>
<td style="text-align: right;">80</td>
<td style="text-align: right;">1.0958904</td>
<td style="text-align: right;">1.0958904</td>
<td style="text-align: right;">23</td>
<td style="text-align: right;">0.1167513</td>
<td style="text-align: right;">230</td>
<td style="text-align: right;">1.1675127</td>
<td style="text-align: right;">1.1675127</td>
<td style="text-align: right;">118.506</td>
<td style="text-align: right;">0.0990223</td>
<td style="text-align: right;">1185.06</td>
<td style="text-align: right;">0.9902228</td>
<td style="text-align: right;">0.9902228</td>
</tr>
<tr class="odd">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.05</td>
<td style="text-align: right;">29</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">14</td>
<td style="text-align: right;">18</td>
<td style="text-align: right;">36.5</td>
<td style="text-align: right;">31.63333</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">65.717</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">0.0821918</td>
<td style="text-align: right;">60</td>
<td style="text-align: right;">0.8219178</td>
<td style="text-align: right;">0.8219178</td>
<td style="text-align: right;">12</td>
<td style="text-align: right;">0.0609137</td>
<td style="text-align: right;">120</td>
<td style="text-align: right;">0.6091371</td>
<td style="text-align: right;">0.6091371</td>
<td style="text-align: right;">65.717</td>
<td style="text-align: right;">0.0549124</td>
<td style="text-align: right;">657.17</td>
<td style="text-align: right;">0.5491238</td>
<td style="text-align: right;">0.5491238</td>
</tr>
<tr class="even">
<td style="text-align: left;">1.FA1</td>
<td style="text-align: left;">1.FA1.06</td>
<td style="text-align: right;">130</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">22</td>
<td style="text-align: right;">30</td>
<td style="text-align: right;">40.3</td>
<td style="text-align: right;">34.35000</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">6</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">9</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">29</td>
<td style="text-align: right;">157.325</td>
<td style="text-align: right;">10</td>
<td style="text-align: right;">73</td>
<td style="text-align: right;">197</td>
<td style="text-align: right;">1196.761</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">0.1095890</td>
<td style="text-align: right;">80</td>
<td style="text-align: right;">1.0958904</td>
<td style="text-align: right;">1.0958904</td>
<td style="text-align: right;">29</td>
<td style="text-align: right;">0.1472081</td>
<td style="text-align: right;">290</td>
<td style="text-align: right;">1.4720812</td>
<td style="text-align: right;">1.4720812</td>
<td style="text-align: right;">157.325</td>
<td style="text-align: right;">0.1314590</td>
<td style="text-align: right;">1573.25</td>
<td style="text-align: right;">1.3145900</td>
<td style="text-align: right;">1.3145900</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Okay now we proceed to ACP.</p>
<section id="acp" class="level3">
<h3 class="anchored" data-anchor-id="acp">ACP</h3>
<div class="grid">
<section id="females" class="level4 g-col-6">
<h4 class="anchored" data-anchor-id="females">Females</h4>
<p><strong>Carry-over 10</strong></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-7-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><strong>Carry-over 5</strong></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-8-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><strong>Carry-over 20</strong></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="males" class="level4 g-col-6">
<h4 class="anchored" data-anchor-id="males">Males</h4>
<p><strong>Carry-over 10</strong></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-10-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><strong>Carry-over 5</strong></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><strong>Carry-over 20</strong></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-12-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
</div>
<section id="both-sexes" class="level4">
<h4 class="anchored" data-anchor-id="both-sexes">Both sexes</h4>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-13-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<blockquote class="blockquote">
<p><strong>The three best based on PCA : index_A3, index_B3 and index A_1. From now on we focus solely on a carry-over of 10, as it’s very similar.</strong></p>
</blockquote>
</section>
</section>
<section id="correlation-test" class="level3">
<h3 class="anchored" data-anchor-id="correlation-test">Correlation test</h3>
<div class="grid">
<section id="females-1" class="level4 g-col-6">
<h4 class="anchored" data-anchor-id="females-1">Females</h4>
<p><strong>A1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_A1
t = 11.381, df = 136, p-value &lt; 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.6015364 0.7750788
sample estimates:
      cor 
0.6984348 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_A1
S = 109846, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.7492027 </code></pre>
</div>
</div>
<p><strong>A2</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_A2
t = -0.52347, df = 136, p-value = 0.6015
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.2103704  0.1231860
sample estimates:
        cor 
-0.04484194 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_A2
S = 395711, p-value = 0.2601
alternative hypothesis: true rho is not equal to 0
sample estimates:
       rho 
0.09652734 </code></pre>
</div>
</div>
<p><strong>A3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_A3
t = 13.653, df = 136, p-value &lt; 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.6796225 0.8229165
sample estimates:
      cor 
0.7603731 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_A3
S = 84479, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.8071208 </code></pre>
</div>
</div>
<p><strong>A4</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_A4
t = 3.2682, df = 136, p-value = 0.00137
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.1075987 0.4181027
sample estimates:
      cor 
0.2698515 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_A4
S = 288311, p-value = 4.092e-05
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.3417384 </code></pre>
</div>
</div>
<p><strong>A5</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_A5
t = -1.9169, df = 136, p-value = 0.05734
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.320614804  0.005041439
sample estimates:
    cor 
-0.1622 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_A5
S = 463225, p-value = 0.5021
alternative hypothesis: true rho is not equal to 0
sample estimates:
        rho 
-0.05761802 </code></pre>
</div>
</div>
<p><strong>B1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_B1
t = 6.6539, df = 136, p-value = 6.414e-10
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.3581291 0.6119994
sample estimates:
      cor 
0.4955762 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_B1
S = 194058, p-value = 1.306e-12
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.5569332 </code></pre>
</div>
</div>
<p><strong>B2</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_B2
t = -0.25705, df = 136, p-value = 0.7975
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.1884475  0.1456044
sample estimates:
       cor 
-0.0220366 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_B2
S = 374554, p-value = 0.0901
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.1448333 </code></pre>
</div>
</div>
<p><strong>B3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_B3
t = 7.394, df = 136, p-value = 1.317e-11
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.4045660 0.6448723
sample estimates:
      cor 
0.5354704 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_B3
S = 160442, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.6336852 </code></pre>
</div>
</div>
<p><strong>B4</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_B4
t = 2.2365, df = 136, p-value = 0.02695
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.02193452 0.34460763
sample estimates:
     cor 
0.188349 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_B4
S = 309817, p-value = 0.0004958
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.2926366 </code></pre>
</div>
</div>
<p><strong>B5</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_B5
t = -1.6294, df = 136, p-value = 0.1056
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.29857309  0.02941289
sample estimates:
       cor 
-0.1383721 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_fem_co10 and data_proxy$dt_proxy$index_B5
S = 434535, p-value = 0.9269
alternative hypothesis: true rho is not equal to 0
sample estimates:
        rho 
0.007885578 </code></pre>
</div>
</div>
</section>
<section id="males-1" class="level4 g-col-6">
<h4 class="anchored" data-anchor-id="males-1">Males</h4>
<p><strong>A1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_A1
t = 12.122, df = 136, p-value &lt; 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.6293332 0.7923391
sample estimates:
      cor 
0.7206511 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_A1
S = 104123, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
    rho 
0.76227 </code></pre>
</div>
</div>
<p><strong>A2</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_A2
t = -0.16192, df = 136, p-value = 0.8716
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.1805694  0.1535777
sample estimates:
        cor 
-0.01388345 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_A2
S = 398429, p-value = 0.2921
alternative hypothesis: true rho is not equal to 0
sample estimates:
       rho 
0.09032237 </code></pre>
</div>
</div>
<p><strong>A3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_A3
t = 14.541, df = 136, p-value &lt; 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.7049039 0.8379812
sample estimates:
      cor 
0.7801166 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_A3
S = 77512, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.8230266 </code></pre>
</div>
</div>
<p><strong>A4</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_A4
t = 3.6489, df = 136, p-value = 0.0003743
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.1384167 0.4435846
sample estimates:
      cor 
0.2986146 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_A4
S = 283390, p-value = 2.173e-05
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.3529735 </code></pre>
</div>
</div>
<p><strong>A5</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_A5
t = -1.8643, df = 136, p-value = 0.06444
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.316610047  0.009498434
sample estimates:
      cor 
-0.157857 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_A5
S = 470381, p-value = 0.3886
alternative hypothesis: true rho is not equal to 0
sample estimates:
        rho 
-0.07395657 </code></pre>
</div>
</div>
<p><strong>B1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_B1
t = 6.6886, df = 136, p-value = 5.368e-10
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.3603790 0.6136126
sample estimates:
      cor 
0.4975225 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_B1
S = 195666, p-value = 1.963e-12
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.5532618 </code></pre>
</div>
</div>
<p><strong>B2</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_B2
t = -0.35112, df = 136, p-value = 0.726
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.1962127  0.1377027
sample estimates:
        cor 
-0.03009462 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_B2
S = 386699, p-value = 0.1714
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.1171035 </code></pre>
</div>
</div>
<p><strong>B3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_B3
t = 7.4052, df = 136, p-value = 1.239e-11
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.4052485 0.6453488
sample estimates:
      cor 
0.5360524 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_B3
S = 160222, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.6341882 </code></pre>
</div>
</div>
<p><strong>B4</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_B4
t = 2.1615, df = 136, p-value = 0.03241
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.0156129 0.3390227
sample estimates:
      cor 
0.1822423 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_B4
S = 319974, p-value = 0.001394
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.2694464 </code></pre>
</div>
</div>
<p><strong>B5</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_B5
t = -1.878, df = 136, p-value = 0.06252
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.31765519  0.00833652
sample estimates:
       cor 
-0.1589898 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy$oms_mal_co10 and data_proxy$dt_proxy$index_B5
S = 446732, p-value = 0.8162
alternative hypothesis: true rho is not equal to 0
sample estimates:
       rho 
-0.0199619 </code></pre>
</div>
</div>
</section>
</div>
<section id="both-sexes-1" class="level4">
<h4 class="anchored" data-anchor-id="both-sexes-1">Both sexes</h4>
<p><strong>A1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_A1
t = 16.657, df = 274, p-value &lt; 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.6452867 0.7634463
sample estimates:
      cor 
0.7093143 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_A1
S = 859377, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.7547476 </code></pre>
</div>
</div>
<p><strong>A2</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_A2
t = -0.48984, df = 274, p-value = 0.6246
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.14713438  0.08880017
sample estimates:
        cor 
-0.02957909 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_A2
S = 3174944, p-value = 0.1195
alternative hypothesis: true rho is not equal to 0
sample estimates:
       rho 
0.09392163 </code></pre>
</div>
</div>
<p><strong>A3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_A3
t = 19.978, df = 274, p-value &lt; 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.7171602 0.8140832
sample estimates:
      cor 
0.7700276 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_A3
S = 651319, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
     rho 
0.814124 </code></pre>
</div>
</div>
<p><strong>A4</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_A4
t = 4.9029, df = 274, p-value = 1.62e-06
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.1716886 0.3890252
sample estimates:
      cor 
0.2840007 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_A4
S = 2286581, p-value = 2.996e-09
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.3474463 </code></pre>
</div>
</div>
<p><strong>A5</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_A5
t = -2.6838, df = 274, p-value = 0.007723
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.27295455 -0.04278249
sample estimates:
       cor 
-0.1600432 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_A5
S = 3732795, p-value = 0.2798
alternative hypothesis: true rho is not equal to 0
sample estimates:
        rho 
-0.06528027 </code></pre>
</div>
</div>
<p><strong>B1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_B1
t = 9.4676, df = 274, p-value &lt; 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.4019805 0.5805247
sample estimates:
      cor 
0.4964857 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_B1
S = 1560081, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.5547777 </code></pre>
</div>
</div>
<p><strong>B2</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_B2
t = -0.43062, df = 274, p-value = 0.6671
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.14363402  0.09234679
sample estimates:
       cor 
-0.0260059 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_B2
S = 3044440, p-value = 0.02936
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.1311653 </code></pre>
</div>
</div>
<p><strong>B3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_B3
t = 10.501, df = 274, p-value &lt; 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.4458333 0.6148813
sample estimates:
      cor 
0.5357035 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_B3
S = 1283586, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
     rho 
0.633685 </code></pre>
</div>
</div>
<p><strong>B4</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_B4
t = 3.1217, df = 274, p-value = 0.001991
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.06875556 0.29689323
sample estimates:
      cor 
0.1853203 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_B4
S = 2519482, p-value = 2.109e-06
alternative hypothesis: true rho is not equal to 0
sample estimates:
    rho 
0.28098 </code></pre>
</div>
</div>
<p><strong>B5</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_B5
t = -2.486, df = 274, p-value = 0.01351
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.26199482 -0.03099428
sample estimates:
       cor 
-0.1485199 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_B5
S = 3522860, p-value = 0.9293
alternative hypothesis: true rho is not equal to 0
sample estimates:
         rho 
-0.005368191 </code></pre>
</div>
</div>
<p><strong>C1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_C1
t = 5.859, df = 274, p-value = 1.333e-08
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.2244416 0.4346157
sample estimates:
      cor 
0.3336687 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_C1
S = 2088552, p-value = 2.944e-12
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.4039604 </code></pre>
</div>
</div>
<p><strong>C2</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_C2
t = -0.39784, df = 274, p-value = 0.6911
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.1416946  0.0943093
sample estimates:
        cor 
-0.02402739 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_C2
S = 3285389, p-value = 0.3016
alternative hypothesis: true rho is not equal to 0
sample estimates:
       rho 
0.06240246 </code></pre>
</div>
</div>
<p><strong>C3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_C3
t = 6.8293, df = 274, p-value = 5.496e-11
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.2757356 0.4779359
sample estimates:
      cor 
0.3813884 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_C3
S = 1832240, p-value &lt; 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.4771079 </code></pre>
</div>
</div>
<p><strong>C4</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_C4
t = 2.2851, df = 274, p-value = 0.02307
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.01898758 0.25077036
sample estimates:
      cor 
0.1367502 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_C4
S = 2810136, p-value = 0.0009402
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.1980319 </code></pre>
</div>
</div>
<p><strong>C5</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
    Pearson's product-moment correlation

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_C5
t = -2.2921, df = 274, p-value = 0.02266
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.25116179 -0.01940517
sample estimates:
       cor 
-0.1371601 </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
    Spearman's rank correlation rho

data:  data_proxy$dt_proxy_longer$oms_co10 and data_proxy$dt_proxy_longer$index_C5
S = 3621980, p-value = 0.5777
alternative hypothesis: true rho is not equal to 0
sample estimates:
        rho 
-0.03365521 </code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>The best proxies : A3 &gt; A1 &gt; B3 &gt; B1. From now we only focus on these indexes, and we just check on females as we have the same results.</strong></p>
</blockquote>
</section>
</section>
<section id="linear-models-and-r2" class="level3">
<h3 class="anchored" data-anchor-id="linear-models-and-r2">Linear models and R<sup>2</sup></h3>
<p><strong>A3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Data: dt_proxy_longer
Models:
linear_models_oms_proxy$mod_A3_null_1: oms_co10 ~ index_A3 + sex + (1 | session) + (1 | session:id)
linear_models_oms_proxy$mod_A3: oms_co10 ~ index_A3 * sex + (1 | session) + (1 | session:id)
                                      npar    AIC    BIC logLik deviance  Chisq
linear_models_oms_proxy$mod_A3_null_1    6 854.81 876.53 -421.4   842.81       
linear_models_oms_proxy$mod_A3           7 856.81 882.15 -421.4   842.81 0.0023
                                      Df Pr(&gt;Chisq)
linear_models_oms_proxy$mod_A3_null_1              
linear_models_oms_proxy$mod_A3         1     0.9621</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: dt_proxy_longer
Models:
linear_models_oms_proxy$mod_A3_null_2: oms_co10 ~ index_A3 + (1 | session) + (1 | session:id)
linear_models_oms_proxy$mod_A3_null_1: oms_co10 ~ index_A3 + sex + (1 | session) + (1 | session:id)
                                      npar    AIC    BIC logLik deviance Chisq
linear_models_oms_proxy$mod_A3_null_2    5 852.81 870.91 -421.4   842.81      
linear_models_oms_proxy$mod_A3_null_1    6 854.81 876.53 -421.4   842.81     0
                                      Df Pr(&gt;Chisq)
linear_models_oms_proxy$mod_A3_null_2              
linear_models_oms_proxy$mod_A3_null_1  1          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_co10 ~ index_A3 + (1 | session) + (1 | session:id)
   Data: dt_proxy_longer

REML criterion at convergence: 852.4

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-2.72576 -0.46200  0.09061  0.55288  2.48496 

Random effects:
 Groups     Name        Variance Std.Dev.
 session:id (Intercept) 0.2215   0.4707  
 session    (Intercept) 2.5691   1.6029  
 Residual               0.8188   0.9049  
Number of obs: 276, groups:  session:id, 138; session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept) 4.166e+00  4.293e-01 1.762e+01   9.706 1.72e-08 ***
index_A3    3.652e-02  3.366e-03 1.271e+02  10.851  &lt; 2e-16 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
         (Intr)
index_A3 -0.320</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.824
     Marginal R2: 0.226</code></pre>
</div>
</div>
<p><strong>A1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Data: dt_proxy_longer
Models:
linear_models_oms_proxy$mod_A1_null_1: oms_co10 ~ index_A1 + sex + (1 | session) + (1 | session:id)
linear_models_oms_proxy$mod_A1: oms_co10 ~ index_A1 * sex + (1 | session) + (1 | session:id)
                                      npar    AIC    BIC  logLik deviance
linear_models_oms_proxy$mod_A1_null_1    6 852.63 874.35 -420.31   840.63
linear_models_oms_proxy$mod_A1           7 854.63 879.97 -420.31   840.63
                                       Chisq Df Pr(&gt;Chisq)
linear_models_oms_proxy$mod_A1_null_1                     
linear_models_oms_proxy$mod_A1        0.0023  1     0.9618</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: dt_proxy_longer
Models:
linear_models_oms_proxy$mod_A1_null_2: oms_co10 ~ index_A1 + (1 | session) + (1 | session:id)
linear_models_oms_proxy$mod_A1_null_1: oms_co10 ~ index_A1 + sex + (1 | session) + (1 | session:id)
                                      npar    AIC    BIC  logLik deviance Chisq
linear_models_oms_proxy$mod_A1_null_2    5 850.63 868.73 -420.31   840.63      
linear_models_oms_proxy$mod_A1_null_1    6 852.63 874.35 -420.31   840.63     0
                                      Df Pr(&gt;Chisq)
linear_models_oms_proxy$mod_A1_null_2              
linear_models_oms_proxy$mod_A1_null_1  1          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_co10 ~ index_A1 + (1 | session) + (1 | session:id)
   Data: dt_proxy_longer

REML criterion at convergence: 845.6

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-2.66842 -0.45841  0.09952  0.57970  2.48953 

Random effects:
 Groups     Name        Variance Std.Dev.
 session:id (Intercept) 0.1952   0.4419  
 session    (Intercept) 3.1201   1.7664  
 Residual               0.8188   0.9049  
Number of obs: 276, groups:  session:id, 138; session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)   4.14978    0.46709  17.43564   8.884 6.93e-08 ***
index_A1      0.33783    0.03057 124.40146  11.050  &lt; 2e-16 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
         (Intr)
index_A1 -0.292</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.837
     Marginal R2: 0.176</code></pre>
</div>
</div>
<p><strong>B3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Data: dt_proxy_longer
Models:
linear_models_oms_proxy$mod_B3_null_1: oms_co10 ~ index_B3 + sex + (1 | session) + (1 | session:id)
linear_models_oms_proxy$mod_B3: oms_co10 ~ index_B3 * sex + (1 | session) + (1 | session:id)
                                      npar    AIC    BIC  logLik deviance
linear_models_oms_proxy$mod_B3_null_1    6 914.65 936.37 -451.32   902.65
linear_models_oms_proxy$mod_B3           7 916.54 941.88 -451.27   902.54
                                       Chisq Df Pr(&gt;Chisq)
linear_models_oms_proxy$mod_B3_null_1                     
linear_models_oms_proxy$mod_B3        0.1103  1     0.7398</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: dt_proxy_longer
Models:
linear_models_oms_proxy$mod_B3_null_2: oms_co10 ~ index_B3 + (1 | session) + (1 | session:id)
linear_models_oms_proxy$mod_B3_null_1: oms_co10 ~ index_B3 + sex + (1 | session) + (1 | session:id)
                                      npar    AIC    BIC  logLik deviance Chisq
linear_models_oms_proxy$mod_B3_null_2    5 912.65 930.75 -451.32   902.65      
linear_models_oms_proxy$mod_B3_null_1    6 914.65 936.37 -451.32   902.65     0
                                      Df Pr(&gt;Chisq)
linear_models_oms_proxy$mod_B3_null_2              
linear_models_oms_proxy$mod_B3_null_1  1          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_co10 ~ index_B3 + (1 | session) + (1 | session:id)
   Data: dt_proxy_longer

REML criterion at convergence: 915.4

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.6069 -0.4378  0.1216  0.5947  2.6681 

Random effects:
 Groups     Name        Variance Std.Dev.
 session:id (Intercept) 0.5585   0.7473  
 session    (Intercept) 4.1445   2.0358  
 Residual               0.8188   0.9049  
Number of obs: 276, groups:  session:id, 138; session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept) 5.087e+00  5.271e-01 1.596e+01   9.650 4.59e-08 ***
index_B3    2.783e-03  5.218e-04 1.247e+02   5.333 4.39e-07 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
         (Intr)
index_B3 -0.203</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.860
     Marginal R2: 0.055</code></pre>
</div>
</div>
<p><strong>B1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Data: dt_proxy_longer
Models:
linear_models_oms_proxy$mod_B1_null_1: oms_co10 ~ index_B1 + sex + (1 | session) + (1 | session:id)
linear_models_oms_proxy$mod_B1: oms_co10 ~ index_B1 * sex + (1 | session) + (1 | session:id)
                                      npar    AIC    BIC  logLik deviance
linear_models_oms_proxy$mod_B1_null_1    6 913.09 934.81 -450.54   901.09
linear_models_oms_proxy$mod_B1           7 915.01 940.35 -450.51   901.01
                                       Chisq Df Pr(&gt;Chisq)
linear_models_oms_proxy$mod_B1_null_1                     
linear_models_oms_proxy$mod_B1        0.0761  1     0.7827</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: dt_proxy_longer
Models:
linear_models_oms_proxy$mod_B1_null_2: oms_co10 ~ index_B1 + (1 | session) + (1 | session:id)
linear_models_oms_proxy$mod_B1_null_1: oms_co10 ~ index_B1 + sex + (1 | session) + (1 | session:id)
                                      npar    AIC    BIC  logLik deviance Chisq
linear_models_oms_proxy$mod_B1_null_2    5 911.09 929.19 -450.54   901.09      
linear_models_oms_proxy$mod_B1_null_1    6 913.09 934.81 -450.54   901.09     0
                                      Df Pr(&gt;Chisq)
linear_models_oms_proxy$mod_B1_null_2              
linear_models_oms_proxy$mod_B1_null_1  1          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_co10 ~ index_B1 + (1 | session) + (1 | session:id)
   Data: dt_proxy_longer

REML criterion at convergence: 909.3

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.5917 -0.4375  0.1234  0.5714  2.6781 

Random effects:
 Groups     Name        Variance Std.Dev.
 session:id (Intercept) 0.5418   0.7361  
 session    (Intercept) 4.2991   2.0734  
 Residual               0.8188   0.9049  
Number of obs: 276, groups:  session:id, 138; session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept) 5.046e+00  5.369e-01 1.611e+01   9.398 6.10e-08 ***
index_B1    2.776e-02  5.041e-03 1.238e+02   5.506 2.02e-07 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
         (Intr)
index_B1 -0.207</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.863
     Marginal R2: 0.052</code></pre>
</div>
</div>
<p><strong>C3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Data: dt_proxy_longer
Models:
linear_models_oms_proxy$mod_C3_null_1: oms_co10 ~ index_C3 + sex + (1 | session) + (1 | session:id)
linear_models_oms_proxy$mod_C3: oms_co10 ~ index_C3 * sex + (1 | session) + (1 | session:id)
                                      npar    AIC    BIC  logLik deviance
linear_models_oms_proxy$mod_C3_null_1    6 927.42 949.14 -457.71   915.42
linear_models_oms_proxy$mod_C3           7 929.39 954.73 -457.69   915.39
                                       Chisq Df Pr(&gt;Chisq)
linear_models_oms_proxy$mod_C3_null_1                     
linear_models_oms_proxy$mod_C3        0.0346  1     0.8525</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: dt_proxy_longer
Models:
linear_models_oms_proxy$mod_C3_null_2: oms_co10 ~ index_C3 + (1 | session) + (1 | session:id)
linear_models_oms_proxy$mod_C3_null_1: oms_co10 ~ index_C3 + sex + (1 | session) + (1 | session:id)
                                      npar    AIC    BIC  logLik deviance Chisq
linear_models_oms_proxy$mod_C3_null_2    5 925.42 943.52 -457.71   915.42      
linear_models_oms_proxy$mod_C3_null_1    6 927.42 949.14 -457.71   915.42     0
                                      Df Pr(&gt;Chisq)
linear_models_oms_proxy$mod_C3_null_2              
linear_models_oms_proxy$mod_C3_null_1  1          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_co10 ~ index_C3 + (1 | session) + (1 | session:id)
   Data: dt_proxy_longer

REML criterion at convergence: 932.6

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.6065 -0.4493  0.1026  0.5665  2.6649 

Random effects:
 Groups     Name        Variance Std.Dev.
 session:id (Intercept) 0.6462   0.8038  
 session    (Intercept) 4.7612   2.1820  
 Residual               0.8188   0.9049  
Number of obs: 276, groups:  session:id, 138; session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept) 5.351e+00  5.589e-01 1.550e+01   9.575 6.63e-08 ***
index_C3    1.998e-04  5.356e-05 1.230e+02   3.730 0.000291 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
         (Intr)
index_C3 -0.147
fit warnings:
Some predictor variables are on very different scales: consider rescaling</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.871
     Marginal R2: 0.022</code></pre>
</div>
</div>
<p><strong>C1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Data: dt_proxy_longer
Models:
linear_models_oms_proxy$mod_C1_null_1: oms_co10 ~ index_C1 + sex + (1 | session) + (1 | session:id)
linear_models_oms_proxy$mod_C1: oms_co10 ~ index_C1 * sex + (1 | session) + (1 | session:id)
                                      npar    AIC    BIC  logLik deviance
linear_models_oms_proxy$mod_C1_null_1    6 926.94 948.66 -457.47   914.94
linear_models_oms_proxy$mod_C1           7 928.93 954.27 -457.46   914.93
                                       Chisq Df Pr(&gt;Chisq)
linear_models_oms_proxy$mod_C1_null_1                     
linear_models_oms_proxy$mod_C1        0.0086  1     0.9262</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: dt_proxy_longer
Models:
linear_models_oms_proxy$mod_C1_null_2: oms_co10 ~ index_C1 + (1 | session) + (1 | session:id)
linear_models_oms_proxy$mod_C1_null_1: oms_co10 ~ index_C1 + sex + (1 | session) + (1 | session:id)
                                      npar    AIC    BIC  logLik deviance Chisq
linear_models_oms_proxy$mod_C1_null_2    5 924.94 943.04 -457.47   914.94      
linear_models_oms_proxy$mod_C1_null_1    6 926.94 948.66 -457.47   914.94     0
                                      Df Pr(&gt;Chisq)
linear_models_oms_proxy$mod_C1_null_2              
linear_models_oms_proxy$mod_C1_null_1  1          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_co10 ~ index_C1 + (1 | session) + (1 | session:id)
   Data: dt_proxy_longer

REML criterion at convergence: 927.6

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.6144 -0.4473  0.1076  0.5690  2.6724 

Random effects:
 Groups     Name        Variance Std.Dev.
 session:id (Intercept) 0.6389   0.7993  
 session    (Intercept) 4.8708   2.2070  
 Residual               0.8188   0.9049  
Number of obs: 276, groups:  session:id, 138; session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept) 5.332e+00  5.654e-01 1.558e+01   9.431 7.75e-08 ***
index_C1    1.962e-03  5.159e-04 1.226e+02   3.803 0.000224 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
         (Intr)
index_C1 -0.151</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.873
     Marginal R2: 0.022</code></pre>
</div>
</div>
<p>Conditionnal R2 (fixed and random effects) :</p>
<ul>
<li>A3 : 0.8243104&nbsp;</li>
<li>A1 : 0.8368409&nbsp;</li>
<li>B3 : 0.8598035&nbsp;</li>
<li>B1 : 0.8628357&nbsp;</li>
<li>C3 : 0.8714332&nbsp;</li>
<li>C1 : 0.8734325&nbsp;</li>
</ul>
<p>Marginal R2 (fixed effects only) :</p>
<ul>
<li>A3 : 0.225549&nbsp;</li>
<li>A1 : 0.1762318&nbsp;</li>
<li>B3 : 0.0545803&nbsp;</li>
<li>B1 : 0.0519268&nbsp;</li>
<li>C3 : 0.0224185&nbsp;</li>
<li>C1 : 0.0217909&nbsp;</li>
</ul>
<blockquote class="blockquote">
<p><strong>Based on marginal, A3 is the best proxy, but based on conditionnal, C1 is better.</strong></p>
</blockquote>
</section>
<section id="aic" class="level3">
<h3 class="anchored" data-anchor-id="aic">AIC</h3>
<p><strong>A3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 878.2016</code></pre>
</div>
</div>
<p><strong>A1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 866.7982</code></pre>
</div>
</div>
<p><strong>B3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 945.1387</code></pre>
</div>
</div>
<p><strong>B1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 934.4144</code></pre>
</div>
</div>
<p><strong>C3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 945.1387</code></pre>
</div>
</div>
<p><strong>C1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] 934.4144</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Best AIC is for A1.</strong></p>
</blockquote>
</section>
<section id="root-mean-squared-error" class="level3">
<h3 class="anchored" data-anchor-id="root-mean-squared-error">Root Mean Squared Error</h3>
<p><strong>A3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] "RMSE: 0.802033678659139"</code></pre>
</div>
</div>
<p><strong>A1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] "RMSE: 0.808217845234384"</code></pre>
</div>
</div>
<p><strong>B3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] "RMSE: 0.749537560258758"</code></pre>
</div>
</div>
<p><strong>B1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] "RMSE: 0.751297290085002"</code></pre>
</div>
</div>
<p><strong>C3</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] "RMSE: 0.741022287263041"</code></pre>
</div>
</div>
<p><strong>C1</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>[1] "RMSE: 0.741665331479704"</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Best based on RMSE is C3.</strong></p>
</blockquote>
</section>
<section id="conclusions" class="level3">
<h3 class="anchored" data-anchor-id="conclusions">Conclusions</h3>
<blockquote class="blockquote">
<p>To summarize, the number of contacts, and the number of contacts x max_oms, seems to be both good predictors of observational mating success. To do : Bateman gradients with the number of contacts to see if we obtain the same results? Necessary?</p>
</blockquote>
</section>
<section id="plots" class="level3">
<h3 class="anchored" data-anchor-id="plots">Plots</h3>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-67-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="which-traits-are-good-predictor-of-mate-access" class="level2">
<h2 class="anchored" data-anchor-id="which-traits-are-good-predictor-of-mate-access">Which traits are good predictor of mate access?</h2>
<p>We measured:</p>
<ul>
<li><p><strong>nb_flo:</strong> the number of opened flowers</p></li>
<li><p><strong>nb_flo_open:</strong> the number of opened flowers (including very new ones)</p></li>
<li><p><strong>nb_flo_all:</strong> the total number of flowers (including very new and old ones)</p></li>
<li><p><strong>height_max:</strong> the height of the highest flower</p></li>
<li><p><strong>height_mean:</strong> the mean height of the highest flowers on each stem</p></li>
</ul>
<p>We analyse the correlation between these traits and the number of contacts,</p>
<p>which had in turn an effect on oms as we saw above.</p>
<section id="simple-linear-models" class="level3">
<h3 class="anchored" data-anchor-id="simple-linear-models">Simple linear models</h3>
<section id="number-of-contacts" class="level4">
<h4 class="anchored" data-anchor-id="number-of-contacts">Number of contacts</h4>
<p>Simple linear models, regression of number of contacts on each trait.</p>
<p><strong>nb_flo</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: index_A1 ~ nb_flo + (1 | session)
   Data: data_proxy$dt_proxy

REML criterion at convergence: 646.5

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.1467 -0.5929 -0.0645  0.5791  4.2087 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 3.221    1.795   
 Residual             5.035    2.244   
Number of obs: 138, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)   3.38887    0.72084  47.97546   4.701 2.21e-05 ***
nb_flo        0.09234    0.04392 135.66896   2.103   0.0373 *  
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
       (Intr)
nb_flo -0.735</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.409
     Marginal R2: 0.031</code></pre>
</div>
</div>
<p><strong>nb_flo_open</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: index_A1 ~ nb_flo_open + (1 | session)
   Data: data_proxy$dt_proxy

REML criterion at convergence: 647.2

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.1711 -0.6088 -0.0663  0.6163  4.1507 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 3.198    1.788   
 Residual             5.053    2.248   
Number of obs: 138, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)   3.45356    0.71310  48.31683   4.843 1.36e-05 ***
nb_flo_open   0.07182    0.03561 135.99881   2.017   0.0457 *  
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
            (Intr)
nb_flo_open -0.730</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.404
     Marginal R2: 0.028</code></pre>
</div>
</div>
<p><strong>nb_flo_all</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: index_A1 ~ nb_flo_all + (1 | session)
   Data: data_proxy$dt_proxy

REML criterion at convergence: 648.1

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.0755 -0.6367 -0.1438  0.5921  4.0907 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 3.272    1.809   
 Residual             5.045    2.246   
Number of obs: 138, groups:  session, 16

Fixed effects:
             Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)   3.28498    0.78258  53.10105   4.198 0.000103 ***
nb_flo_all    0.04793    0.02396 132.55053   2.000 0.047513 *  
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
           (Intr)
nb_flo_all -0.777</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.412
     Marginal R2: 0.031</code></pre>
</div>
</div>
<p><strong>height_max</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: index_A1 ~ height_max + (1 | session)
   Data: data_proxy$dt_proxy

REML criterion at convergence: 658

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.2822 -0.6331 -0.1178  0.6489  4.2839 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 2.843    1.686   
 Residual             5.219    2.285   
Number of obs: 138, groups:  session, 16

Fixed effects:
              Estimate Std. Error         df t value Pr(&gt;|t|)    
(Intercept)  4.560e+00  4.677e-01  1.476e+01   9.751 8.05e-08 ***
height_max  -7.015e-04  5.975e-04  1.295e+02  -1.174    0.243    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
           (Intr)
height_max -0.093</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.357
     Marginal R2: 0.007</code></pre>
</div>
</div>
<p><strong>height_mean</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: index_A1 ~ height_mean + (1 | session)
   Data: data_proxy$dt_proxy

REML criterion at convergence: 655.3

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.2787 -0.6326 -0.1188  0.6486  4.2767 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 2.849    1.688   
 Residual             5.225    2.286   
Number of obs: 138, groups:  session, 16

Fixed effects:
              Estimate Std. Error         df t value Pr(&gt;|t|)    
(Intercept)   4.610338   0.475040  15.583025   9.705 5.27e-08 ***
height_mean  -0.002655   0.002404 129.745261  -1.104    0.272    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
            (Intr)
height_mean -0.193</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.357
     Marginal R2: 0.007</code></pre>
</div>
</div>
<p><strong>all traits together</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: index_A1 ~ nb_flo_open + height_max + (1 | session)
   Data: data_proxy$dt_proxy

REML criterion at convergence: 658.6

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.1693 -0.5997 -0.0423  0.6137  4.1527 

Random effects:
 Groups   Name        Variance Std.Dev.
 session  (Intercept) 3.129    1.769   
 Residual             5.038    2.244   
Number of obs: 138, groups:  session, 16

Fixed effects:
              Estimate Std. Error         df t value Pr(&gt;|t|)    
(Intercept)  3.481e+00  7.094e-01  4.857e+01   4.907 1.08e-05 ***
nb_flo_open  7.373e-02  3.557e-02  1.350e+02   2.073   0.0401 *  
height_max  -7.535e-04  5.889e-04  1.276e+02  -1.280   0.2030    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
            (Intr) nb_fl_
nb_flo_open -0.730       
height_max  -0.026 -0.046</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.404
     Marginal R2: 0.034</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>For simple linear models, only flower number have an effect on the number of contacts, and the strongest effect seems for the number of opened flowers (without very new or old ones). When working with all traits together, only flower number has an effect.</strong></p>
</blockquote>
</section>
<section id="observational-mating-success" class="level4">
<h4 class="anchored" data-anchor-id="observational-mating-success">Observational mating success</h4>
<p>Simple linear models, regression of observational mating success on each trait.</p>
<p><strong>nb_flo</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_null_0: oms_co10 ~ nb_flo * sex + sex * ttt + nb_flo * ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo: oms_co10 ~ nb_flo * sex * ttt + (1 | session) + (1 | session:id)
                      npar    AIC    BIC  logLik deviance  Chisq Df Pr(&gt;Chisq)
mod_oms_nb_flo_null_0   13 1169.2 1218.2 -571.62   1143.2                     
mod_oms_nb_flo          15 1173.0 1229.5 -571.47   1143.0 0.2902  2     0.8649</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_null_1: oms_co10 ~ nb_flo + sex + sex * ttt + nb_flo * ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_null_0: oms_co10 ~ nb_flo * sex + sex * ttt + nb_flo * ttt + (1 | session) + (1 | session:id)
                      npar    AIC    BIC  logLik deviance  Chisq Df Pr(&gt;Chisq)
mod_oms_nb_flo_null_1   12 1167.5 1212.7 -571.75   1143.5                     
mod_oms_nb_flo_null_0   13 1169.2 1218.2 -571.62   1143.2 0.2723  1     0.6018</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_null_2: oms_co10 ~ nb_flo + sex + sex + ttt + nb_flo * ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_null_1: oms_co10 ~ nb_flo + sex + sex * ttt + nb_flo * ttt + (1 | session) + (1 | session:id)
                      npar    AIC    BIC  logLik deviance Chisq Df Pr(&gt;Chisq)
mod_oms_nb_flo_null_2   10 1163.5 1201.2 -571.75   1143.5                    
mod_oms_nb_flo_null_1   12 1167.5 1212.7 -571.75   1143.5     0  2          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_null_3: oms_co10 ~ nb_flo + sex + ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_null_2: oms_co10 ~ nb_flo + sex + sex + ttt + nb_flo * ttt + (1 | session) + (1 | session:id)
                      npar    AIC    BIC  logLik deviance  Chisq Df Pr(&gt;Chisq)
mod_oms_nb_flo_null_3    8 1164.1 1194.2 -574.05   1148.1                     
mod_oms_nb_flo_null_2   10 1163.5 1201.2 -571.75   1143.5 4.5966  2     0.1004</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_null_4: oms_co10 ~ nb_flo + sex + (1 | session) + (1 | session:id)
mod_oms_nb_flo_null_3: oms_co10 ~ nb_flo + sex + ttt + (1 | session) + (1 | session:id)
                      npar    AIC    BIC  logLik deviance  Chisq Df Pr(&gt;Chisq)
mod_oms_nb_flo_null_4    6 1168.9 1191.5 -578.45   1156.9                     
mod_oms_nb_flo_null_3    8 1164.1 1194.2 -574.05   1148.1 8.7937  2    0.01232
                       
mod_oms_nb_flo_null_4  
mod_oms_nb_flo_null_3 *
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_null_5: oms_co10 ~ nb_flo + ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_null_3: oms_co10 ~ nb_flo + sex + ttt + (1 | session) + (1 | session:id)
                      npar    AIC    BIC  logLik deviance Chisq Df Pr(&gt;Chisq)
mod_oms_nb_flo_null_5    7 1162.1 1188.5 -574.05   1148.1                    
mod_oms_nb_flo_null_3    8 1164.1 1194.2 -574.05   1148.1     0  1          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_null_6: oms_co10 ~ ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_null_5: oms_co10 ~ nb_flo + ttt + (1 | session) + (1 | session:id)
                      npar    AIC    BIC  logLik deviance  Chisq Df Pr(&gt;Chisq)
mod_oms_nb_flo_null_6    6 1161.0 1183.7 -574.52   1149.0                     
mod_oms_nb_flo_null_5    7 1162.1 1188.5 -574.05   1148.1 0.9314  1     0.3345</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_null_7: oms_co10 ~ nb_flo + (1 | session) + (1 | session:id)
mod_oms_nb_flo_null_5: oms_co10 ~ nb_flo + ttt + (1 | session) + (1 | session:id)
                      npar    AIC    BIC  logLik deviance  Chisq Df Pr(&gt;Chisq)
mod_oms_nb_flo_null_7    5 1166.9 1185.7 -578.45   1156.9                     
mod_oms_nb_flo_null_5    7 1162.1 1188.5 -574.05   1148.1 8.7937  2    0.01232
                       
mod_oms_nb_flo_null_7  
mod_oms_nb_flo_null_5 *
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_co10 ~ ttt + (1 | session) + (1 | session:id)
   Data: data_proxy$dt_proxy_longer

REML criterion at convergence: 1144

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-2.33968 -0.48780  0.07342  0.52734  2.62811 

Random effects:
 Groups     Name        Variance Std.Dev.
 session:id (Intercept) 2.0495   1.4316  
 session    (Intercept) 4.8076   2.1926  
 Residual               0.7063   0.8404  
Number of obs: 320, groups:  session:id, 160; session, 16

Fixed effects:
            Estimate Std. Error     df t value Pr(&gt;|t|)    
(Intercept)    7.550      1.123 13.003   6.721 1.42e-05 ***
tttlow        -4.367      1.450 13.003  -3.011    0.010 *  
tttmedium     -1.917      1.450 13.003  -1.322    0.209    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
          (Intr) tttlow
tttlow    -0.775       
tttmedium -0.775  0.600</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.933
     Marginal R2: 0.283</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Only an effect of treatment.</strong></p>
</blockquote>
<p><strong>nb_flo_open</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_open_null_0: oms_co10 ~ nb_flo_open * sex + sex * ttt + nb_flo_open * ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_open: oms_co10 ~ nb_flo_open * sex * ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_nb_flo_open_null_0   13 1168.2 1217.2 -571.12   1142.2          
mod_oms_nb_flo_open          15 1172.0 1228.5 -571.01   1142.0 0.2208  2
                           Pr(&gt;Chisq)
mod_oms_nb_flo_open_null_0           
mod_oms_nb_flo_open            0.8955</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_open_null_1: oms_co10 ~ nb_flo_open + sex + sex * ttt + nb_flo_open * ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_open_null_0: oms_co10 ~ nb_flo_open * sex + sex * ttt + nb_flo_open * ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_nb_flo_open_null_1   12 1167.0 1212.2 -571.52   1143.0          
mod_oms_nb_flo_open_null_0   13 1168.2 1217.2 -571.12   1142.2 0.7848  1
                           Pr(&gt;Chisq)
mod_oms_nb_flo_open_null_1           
mod_oms_nb_flo_open_null_0     0.3757</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_open_null_2: oms_co10 ~ nb_flo_open + sex + sex + ttt + nb_flo_open * ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_open_null_1: oms_co10 ~ nb_flo_open + sex + sex * ttt + nb_flo_open * ttt + (1 | session) + (1 | session:id)
                           npar  AIC    BIC  logLik deviance Chisq Df
mod_oms_nb_flo_open_null_2   10 1163 1200.7 -571.52     1143         
mod_oms_nb_flo_open_null_1   12 1167 1212.2 -571.52     1143     0  2
                           Pr(&gt;Chisq)
mod_oms_nb_flo_open_null_2           
mod_oms_nb_flo_open_null_1          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_open_null_3: oms_co10 ~ nb_flo_open + sex + ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_open_null_2: oms_co10 ~ nb_flo_open + sex + sex + ttt + nb_flo_open * ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_nb_flo_open_null_3    8 1164.1 1194.3 -574.06   1148.1          
mod_oms_nb_flo_open_null_2   10 1163.0 1200.7 -571.52   1143.0 5.0822  2
                           Pr(&gt;Chisq)  
mod_oms_nb_flo_open_null_3             
mod_oms_nb_flo_open_null_2    0.07878 .
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_open_null_4: oms_co10 ~ nb_flo_open + sex + (1 | session) + (1 | session:id)
mod_oms_nb_flo_open_null_3: oms_co10 ~ nb_flo_open + sex + ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_nb_flo_open_null_4    6 1168.9 1191.5 -578.46   1156.9          
mod_oms_nb_flo_open_null_3    8 1164.1 1194.3 -574.06   1148.1 8.8032  2
                           Pr(&gt;Chisq)  
mod_oms_nb_flo_open_null_4             
mod_oms_nb_flo_open_null_3    0.01226 *
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_open_null_5: oms_co10 ~ nb_flo_open + ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_open_null_3: oms_co10 ~ nb_flo_open + sex + ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance Chisq Df
mod_oms_nb_flo_open_null_5    7 1162.1 1188.5 -574.06   1148.1         
mod_oms_nb_flo_open_null_3    8 1164.1 1194.3 -574.06   1148.1     0  1
                           Pr(&gt;Chisq)
mod_oms_nb_flo_open_null_5           
mod_oms_nb_flo_open_null_3          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_open_null_6: oms_co10 ~ ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_open_null_5: oms_co10 ~ nb_flo_open + ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_nb_flo_open_null_6    6 1161.0 1183.7 -574.52   1149.0          
mod_oms_nb_flo_open_null_5    7 1162.1 1188.5 -574.06   1148.1 0.9228  1
                           Pr(&gt;Chisq)
mod_oms_nb_flo_open_null_6           
mod_oms_nb_flo_open_null_5     0.3367</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_open_null_7: oms_co10 ~ nb_flo_open + (1 | session) + (1 | session:id)
mod_oms_nb_flo_open_null_5: oms_co10 ~ nb_flo_open + ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_nb_flo_open_null_7    5 1166.9 1185.8 -578.46   1156.9          
mod_oms_nb_flo_open_null_5    7 1162.1 1188.5 -574.06   1148.1 8.8032  2
                           Pr(&gt;Chisq)  
mod_oms_nb_flo_open_null_7             
mod_oms_nb_flo_open_null_5    0.01226 *
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_co10 ~ ttt + (1 | session) + (1 | session:id)
   Data: data_proxy$dt_proxy_longer

REML criterion at convergence: 1144

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-2.33968 -0.48780  0.07342  0.52734  2.62811 

Random effects:
 Groups     Name        Variance Std.Dev.
 session:id (Intercept) 2.0495   1.4316  
 session    (Intercept) 4.8076   2.1926  
 Residual               0.7063   0.8404  
Number of obs: 320, groups:  session:id, 160; session, 16

Fixed effects:
            Estimate Std. Error     df t value Pr(&gt;|t|)    
(Intercept)    7.550      1.123 13.003   6.721 1.42e-05 ***
tttlow        -4.367      1.450 13.003  -3.011    0.010 *  
tttmedium     -1.917      1.450 13.003  -1.322    0.209    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
          (Intr) tttlow
tttlow    -0.775       
tttmedium -0.775  0.600</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.933
     Marginal R2: 0.283</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Only an effect of treatment.</strong></p>
</blockquote>
<p><strong>nb_flo_all</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_all_null_0: oms_co10 ~ nb_flo_all * sex + sex * ttt + nb_flo_all * ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_all: oms_co10 ~ nb_flo_all * sex * ttt + (1 | session) + (1 | session:id)
                          npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_nb_flo_all_null_0   13 1168.1 1217.1 -571.07   1142.1          
mod_oms_nb_flo_all          15 1172.0 1228.6 -571.02   1142.0 0.0995  2
                          Pr(&gt;Chisq)
mod_oms_nb_flo_all_null_0           
mod_oms_nb_flo_all            0.9515</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_all_null_1: oms_co10 ~ nb_flo_all + sex + sex * ttt + nb_flo_all * ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_all_null_0: oms_co10 ~ nb_flo_all * sex + sex * ttt + nb_flo_all * ttt + (1 | session) + (1 | session:id)
                          npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_nb_flo_all_null_1   12 1166.2 1211.4 -571.10   1142.2          
mod_oms_nb_flo_all_null_0   13 1168.1 1217.1 -571.07   1142.1 0.0554  1
                          Pr(&gt;Chisq)
mod_oms_nb_flo_all_null_1           
mod_oms_nb_flo_all_null_0     0.8139</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_all_null_2: oms_co10 ~ nb_flo_all + sex + sex + ttt + nb_flo_all * ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_all_null_1: oms_co10 ~ nb_flo_all + sex + sex * ttt + nb_flo_all * ttt + (1 | session) + (1 | session:id)
                          npar    AIC    BIC logLik deviance Chisq Df
mod_oms_nb_flo_all_null_2   10 1162.2 1199.9 -571.1   1142.2         
mod_oms_nb_flo_all_null_1   12 1166.2 1211.4 -571.1   1142.2     0  2
                          Pr(&gt;Chisq)
mod_oms_nb_flo_all_null_2           
mod_oms_nb_flo_all_null_1          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_nb_flo_all_null_4: oms_co10 ~ nb_flo_all * ttt + (1 | session) + (1 | session:id)
mod_oms_nb_flo_all_null_3: oms_co10 ~ sex + nb_flo_all * ttt + (1 | session) + (1 | session:id)
                          npar    AIC    BIC logLik deviance Chisq Df
mod_oms_nb_flo_all_null_4    9 1160.2 1194.1 -571.1   1142.2         
mod_oms_nb_flo_all_null_3   10 1162.2 1199.9 -571.1   1142.2     0  1
                          Pr(&gt;Chisq)
mod_oms_nb_flo_all_null_4           
mod_oms_nb_flo_all_null_3          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_co10 ~ nb_flo_all * ttt + (1 | session) + (1 | session:id)
   Data: data_proxy$dt_proxy_longer

REML criterion at convergence: 1152.9

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-2.41351 -0.48981  0.06771  0.52604  2.59743 

Random effects:
 Groups     Name        Variance Std.Dev.
 session:id (Intercept) 1.9551   1.3983  
 session    (Intercept) 5.4947   2.3441  
 Residual               0.7062   0.8404  
Number of obs: 320, groups:  session:id, 160; session, 16

Fixed effects:
                      Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)            7.02169    1.39911  23.31174   5.019 4.29e-05 ***
nb_flo_all             0.01867    0.02563 142.06223   0.728   0.4676    
tttlow                -2.52005    1.84206  24.52818  -1.368   0.1837    
tttmedium             -2.65832    1.84737  24.57211  -1.439   0.1628    
nb_flo_all:tttlow     -0.06940    0.03701 149.57345  -1.875   0.0627 .  
nb_flo_all:tttmedium   0.03377    0.03887 151.84509   0.869   0.3863    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
              (Intr) nb_fl_ tttlow tttmdm nb_fl_ll:tttl
nb_flo_all    -0.518                                   
tttlow        -0.760  0.394                            
tttmedium     -0.757  0.393  0.575                     
nb_fl_ll:tttl  0.359 -0.692 -0.544 -0.272              
nb_fl_ll:tttm  0.342 -0.659 -0.260 -0.547  0.457       </code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.938
     Marginal R2: 0.286</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>An effect of treatment and of the interaction with flowers.</strong></p>
</blockquote>
<p><strong>height_max</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_max_null_0: oms_co10 ~ height_max * sex + sex * ttt + height_max * ttt + (1 | session) + (1 | session:id)
mod_oms_height_max: oms_co10 ~ height_max * sex * ttt + (1 | session) + (1 | session:id)
                          npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_height_max_null_0   13 1168.4 1217.4 -571.20   1142.4          
mod_oms_height_max          15 1172.1 1228.6 -571.03   1142.1 0.3374  2
                          Pr(&gt;Chisq)
mod_oms_height_max_null_0           
mod_oms_height_max            0.8447</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_max_null_1: oms_co10 ~ height_max + sex + sex * ttt + height_max * ttt + (1 | session) + (1 | session:id)
mod_oms_height_max_null_0: oms_co10 ~ height_max * sex + sex * ttt + height_max * ttt + (1 | session) + (1 | session:id)
                          npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_height_max_null_1   12 1169.3 1214.5 -572.63   1145.3          
mod_oms_height_max_null_0   13 1168.4 1217.4 -571.20   1142.4 2.8599  1
                          Pr(&gt;Chisq)  
mod_oms_height_max_null_1             
mod_oms_height_max_null_0    0.09081 .
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_max_null_2: oms_co10 ~ height_max + sex + sex + ttt + height_max * ttt + (1 | session) + (1 | session:id)
mod_oms_height_max_null_1: oms_co10 ~ height_max + sex + sex * ttt + height_max * ttt + (1 | session) + (1 | session:id)
                          npar    AIC    BIC  logLik deviance Chisq Df
mod_oms_height_max_null_2   10 1165.3 1202.9 -572.63   1145.3         
mod_oms_height_max_null_1   12 1169.3 1214.5 -572.63   1145.3     0  2
                          Pr(&gt;Chisq)
mod_oms_height_max_null_2           
mod_oms_height_max_null_1          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_max_null_3: oms_co10 ~ height_max + sex + ttt + (1 | session) + (1 | session:id)
mod_oms_height_max_null_2: oms_co10 ~ height_max + sex + sex + ttt + height_max * ttt + (1 | session) + (1 | session:id)
                          npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_height_max_null_3    8 1164.7 1194.9 -574.36   1148.7          
mod_oms_height_max_null_2   10 1165.3 1202.9 -572.63   1145.3 3.4586  2
                          Pr(&gt;Chisq)
mod_oms_height_max_null_3           
mod_oms_height_max_null_2     0.1774</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_max_null_4: oms_co10 ~ height_max + sex + (1 | session) + (1 | session:id)
mod_oms_height_max_null_3: oms_co10 ~ height_max + sex + ttt + (1 | session) + (1 | session:id)
                          npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_height_max_null_4    6 1169.4 1192.0 -578.71   1157.4          
mod_oms_height_max_null_3    8 1164.7 1194.9 -574.36   1148.7 8.7014  2
                          Pr(&gt;Chisq)  
mod_oms_height_max_null_4             
mod_oms_height_max_null_3     0.0129 *
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_max_null_5: oms_co10 ~ height_max + ttt + (1 | session) + (1 | session:id)
mod_oms_height_max_null_3: oms_co10 ~ height_max + sex + ttt + (1 | session) + (1 | session:id)
                          npar    AIC    BIC  logLik deviance Chisq Df
mod_oms_height_max_null_5    7 1162.7 1189.1 -574.36   1148.7         
mod_oms_height_max_null_3    8 1164.7 1194.9 -574.36   1148.7     0  1
                          Pr(&gt;Chisq)
mod_oms_height_max_null_5           
mod_oms_height_max_null_3          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_max_null_6: oms_co10 ~ ttt + (1 | session) + (1 | session:id)
mod_oms_height_max_null_5: oms_co10 ~ height_max + ttt + (1 | session) + (1 | session:id)
                          npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_height_max_null_6    6 1161.0 1183.7 -574.52   1149.0          
mod_oms_height_max_null_5    7 1162.7 1189.1 -574.36   1148.7 0.3198  1
                          Pr(&gt;Chisq)
mod_oms_height_max_null_6           
mod_oms_height_max_null_5     0.5717</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_max_null_7: oms_co10 ~ height_max + (1 | session) + (1 | session:id)
mod_oms_height_max_null_5: oms_co10 ~ height_max + ttt + (1 | session) + (1 | session:id)
                          npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_height_max_null_7    5 1167.4 1186.3 -578.71   1157.4          
mod_oms_height_max_null_5    7 1162.7 1189.1 -574.36   1148.7 8.7014  2
                          Pr(&gt;Chisq)  
mod_oms_height_max_null_7             
mod_oms_height_max_null_5     0.0129 *
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_co10 ~ height_max + ttt + (1 | session) + (1 | session:id)
   Data: data_proxy$dt_proxy_longer

REML criterion at convergence: 1157.5

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-2.34021 -0.48812  0.07288  0.52508  2.62696 

Random effects:
 Groups     Name        Variance Std.Dev.
 session:id (Intercept) 2.0620   1.4360  
 session    (Intercept) 4.7910   2.1888  
 Residual               0.7062   0.8404  
Number of obs: 320, groups:  session:id, 160; session, 16

Fixed effects:
              Estimate Std. Error         df t value Pr(&gt;|t|)    
(Intercept)  7.558e+00  1.122e+00  1.300e+01   6.737 1.39e-05 ***
height_max  -2.212e-04  3.988e-04  1.443e+02  -0.555   0.5800    
tttlow      -4.352e+00  1.448e+00  1.301e+01  -3.005   0.0101 *  
tttmedium   -1.917e+00  1.448e+00  1.300e+01  -1.324   0.2083    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
           (Intr) hght_m tttlow
height_max -0.012              
tttlow     -0.774 -0.018       
tttmedium  -0.775  0.001  0.600</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.933
     Marginal R2: 0.283</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Only an effect of treatment.</strong></p>
</blockquote>
<p><strong>height_mean</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_mean_null_0: oms_co10 ~ height_mean * sex + sex * ttt + height_mean * ttt + (1 | session) + (1 | session:id)
mod_oms_height_mean: oms_co10 ~ height_mean * sex * ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_height_mean_null_0   13 1171.0 1220.0 -572.53   1145.0          
mod_oms_height_mean          15 1173.7 1230.2 -571.83   1143.7 1.3925  2
                           Pr(&gt;Chisq)
mod_oms_height_mean_null_0           
mod_oms_height_mean            0.4985</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_mean_null_1: oms_co10 ~ height_mean + sex + sex * ttt + height_mean * ttt + (1 | session) + (1 | session:id)
mod_oms_height_mean_null_0: oms_co10 ~ height_mean * sex + sex * ttt + height_mean * ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance Chisq Df
mod_oms_height_mean_null_1   12 1171.7 1216.9 -573.86   1147.7         
mod_oms_height_mean_null_0   13 1171.0 1220.0 -572.53   1145.0 2.663  1
                           Pr(&gt;Chisq)
mod_oms_height_mean_null_1           
mod_oms_height_mean_null_0     0.1027</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_mean_null_2: oms_co10 ~ height_mean + sex + sex + ttt + height_mean * ttt + (1 | session) + (1 | session:id)
mod_oms_height_mean_null_1: oms_co10 ~ height_mean + sex + sex * ttt + height_mean * ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance Chisq Df
mod_oms_height_mean_null_2   10 1167.7 1205.4 -573.86   1147.7         
mod_oms_height_mean_null_1   12 1171.7 1216.9 -573.86   1147.7     0  2
                           Pr(&gt;Chisq)
mod_oms_height_mean_null_2           
mod_oms_height_mean_null_1          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_mean_null_3: oms_co10 ~ height_mean + sex + ttt + (1 | session) + (1 | session:id)
mod_oms_height_mean_null_2: oms_co10 ~ height_mean + sex + sex + ttt + height_mean * ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_height_mean_null_3    8 1164.8 1194.9 -574.39   1148.8          
mod_oms_height_mean_null_2   10 1167.7 1205.4 -573.86   1147.7 1.0689  2
                           Pr(&gt;Chisq)
mod_oms_height_mean_null_3           
mod_oms_height_mean_null_2      0.586</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_mean_null_4: oms_co10 ~ height_mean + sex + (1 | session) + (1 | session:id)
mod_oms_height_mean_null_3: oms_co10 ~ height_mean + sex + ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_height_mean_null_4    6 1169.5 1192.1 -578.74   1157.5          
mod_oms_height_mean_null_3    8 1164.8 1194.9 -574.39   1148.8 8.7015  2
                           Pr(&gt;Chisq)  
mod_oms_height_mean_null_4             
mod_oms_height_mean_null_3     0.0129 *
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_mean_null_5: oms_co10 ~ height_mean + ttt + (1 | session) + (1 | session:id)
mod_oms_height_mean_null_3: oms_co10 ~ height_mean + sex + ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance Chisq Df
mod_oms_height_mean_null_5    7 1162.8 1189.2 -574.39   1148.8         
mod_oms_height_mean_null_3    8 1164.8 1194.9 -574.39   1148.8     0  1
                           Pr(&gt;Chisq)
mod_oms_height_mean_null_5           
mod_oms_height_mean_null_3          1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_mean_null_6: oms_co10 ~ ttt + (1 | session) + (1 | session:id)
mod_oms_height_mean_null_5: oms_co10 ~ height_mean + ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_height_mean_null_6    6 1161.0 1183.7 -574.52   1149.0          
mod_oms_height_mean_null_5    7 1162.8 1189.2 -574.39   1148.8 0.2537  1
                           Pr(&gt;Chisq)
mod_oms_height_mean_null_6           
mod_oms_height_mean_null_5     0.6145</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_height_mean_null_7: oms_co10 ~ height_mean + (1 | session) + (1 | session:id)
mod_oms_height_mean_null_5: oms_co10 ~ height_mean + ttt + (1 | session) + (1 | session:id)
                           npar    AIC    BIC  logLik deviance  Chisq Df
mod_oms_height_mean_null_7    5 1167.5 1186.3 -578.74   1157.5          
mod_oms_height_mean_null_5    7 1162.8 1189.2 -574.39   1148.8 8.7015  2
                           Pr(&gt;Chisq)  
mod_oms_height_mean_null_7             
mod_oms_height_mean_null_5     0.0129 *
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_co10 ~ height_mean + ttt + (1 | session) + (1 | session:id)
   Data: data_proxy$dt_proxy_longer

REML criterion at convergence: 1154.8

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-2.34002 -0.48821  0.07283  0.52534  2.62707 

Random effects:
 Groups     Name        Variance Std.Dev.
 session:id (Intercept) 2.0628   1.4362  
 session    (Intercept) 4.7961   2.1900  
 Residual               0.7062   0.8404  
Number of obs: 320, groups:  session:id, 160; session, 16

Fixed effects:
              Estimate Std. Error         df t value Pr(&gt;|t|)    
(Intercept)  7.573e+00  1.123e+00  1.304e+01   6.742 1.36e-05 ***
height_mean -7.929e-04  1.603e-03  1.444e+02  -0.495   0.6216    
tttlow      -4.354e+00  1.449e+00  1.301e+01  -3.005   0.0101 *  
tttmedium   -1.917e+00  1.449e+00  1.300e+01  -1.323   0.2086    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Correlation of Fixed Effects:
            (Intr) hght_m tttlow
height_mean -0.041              
tttlow      -0.773 -0.017       
tttmedium   -0.774  0.001  0.600</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.933
     Marginal R2: 0.283</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Only an effect of treatment.</strong></p>
</blockquote>
<p><strong>all traits together</strong></p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_all_null_0: oms_co10 ~ nb_flo_open * ttt + height_max * ttt + (1 | session) + (1 | session:id)
mod_oms_all: oms_co10 ~ nb_flo_open * sex * ttt + height_max * sex * ttt + (1 | session) + (1 | session:id)
                   npar    AIC    BIC  logLik deviance  Chisq Df Pr(&gt;Chisq)
mod_oms_all_null_0   12 1163.8 1209.0 -569.91   1139.8                     
mod_oms_all          21 1176.7 1255.8 -567.34   1134.7 5.1305  9     0.8228</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_all_null_1: oms_co10 ~ nb_flo_open + ttt + height_max * ttt + (1 | session) + (1 | session:id)
mod_oms_all_null_0: oms_co10 ~ nb_flo_open * ttt + height_max * ttt + (1 | session) + (1 | session:id)
                   npar    AIC    BIC  logLik deviance  Chisq Df Pr(&gt;Chisq)  
mod_oms_all_null_1   10 1164.8 1202.5 -572.39   1144.8                       
mod_oms_all_null_0   12 1163.8 1209.0 -569.91   1139.8 4.9682  2     0.0834 .
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_all_null_2: oms_co10 ~ nb_flo_open + ttt + height_max + ttt + (1 | session) + (1 | session:id)
mod_oms_all_null_1: oms_co10 ~ nb_flo_open + ttt + height_max * ttt + (1 | session) + (1 | session:id)
                   npar    AIC    BIC  logLik deviance  Chisq Df Pr(&gt;Chisq)
mod_oms_all_null_2    8 1163.8 1194.0 -573.91   1147.8                     
mod_oms_all_null_1   10 1164.8 1202.5 -572.39   1144.8 3.0375  2      0.219</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_all_null_3: oms_co10 ~ height_max + ttt + (1 | session) + (1 | session:id)
mod_oms_all_null_2: oms_co10 ~ nb_flo_open + ttt + height_max + ttt + (1 | session) + (1 | session:id)
                   npar    AIC    BIC  logLik deviance  Chisq Df Pr(&gt;Chisq)
mod_oms_all_null_3    7 1162.7 1189.1 -574.36   1148.7                     
mod_oms_all_null_2    8 1163.8 1194.0 -573.91   1147.8 0.9001  1     0.3427</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_all_null_4: oms_co10 ~ ttt + (1 | session) + (1 | session:id)
mod_oms_all_null_3: oms_co10 ~ height_max + ttt + (1 | session) + (1 | session:id)
                   npar    AIC    BIC  logLik deviance  Chisq Df Pr(&gt;Chisq)
mod_oms_all_null_4    6 1161.0 1183.7 -574.52   1149.0                     
mod_oms_all_null_3    7 1162.7 1189.1 -574.36   1148.7 0.3198  1     0.5717</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Data: data_proxy$dt_proxy_longer
Models:
mod_oms_all_null_5: oms_co10 ~ 1 + (1 | session) + (1 | session:id)
mod_oms_all_null_3: oms_co10 ~ height_max + ttt + (1 | session) + (1 | session:id)
                   npar    AIC    BIC  logLik deviance  Chisq Df Pr(&gt;Chisq)  
mod_oms_all_null_5    4 1165.8 1180.8 -578.88   1157.8                       
mod_oms_all_null_3    7 1162.7 1189.1 -574.36   1148.7 9.0484  3    0.02865 *
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: oms_co10 ~ 1 + (1 | session) + (1 | session:id)
   Data: data_proxy$dt_proxy_longer

REML criterion at convergence: 1156.7

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-2.34684 -0.48628  0.06108  0.53461  2.61582 

Random effects:
 Groups     Name        Variance Std.Dev.
 session:id (Intercept) 2.0497   1.4317  
 session    (Intercept) 7.3097   2.7036  
 Residual               0.7063   0.8404  
Number of obs: 320, groups:  session:id, 160; session, 16

Fixed effects:
            Estimate Std. Error      df t value Pr(&gt;|t|)    
(Intercept)   5.1937     0.6869 15.0000   7.561 1.71e-06 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.930
     Marginal R2: 0.000</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Concerning the regression of the observational mating success on phenotypic traits, we do not detect any effect (both for simple linear models or when working with all traits together), we only have an effect of treatment.</strong></p>
</blockquote>
</section>
</section>
<section id="sem" class="level3">
<h3 class="anchored" data-anchor-id="sem">SEM</h3>
<p>Best way to do this analysis: SEM?</p>
<section id="complete-sessions-with-all-genotypes" class="level4">
<h4 class="anchored" data-anchor-id="complete-sessions-with-all-genotypes">Complete sessions with all genotypes</h4>
<p>For the medium treatment :</p>
<div class="cell">
<div class="cell-output-display">
<div class="grViz html-widget html-fill-item" id="htmlwidget-6e9746bc9054f2281d94" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-6e9746bc9054f2281d94">{"x":{"diagram":"digraph {\n\ngraph [layout = \"neato\",\n       outputorder = \"edgesfirst\",\n       bgcolor = \"white\"]\n\nnode [fontname = \"Helvetica\",\n      fontsize = \"10\",\n      shape = \"circle\",\n      fixedsize = \"true\",\n      width = \"0.5\",\n      style = \"filled\",\n      fillcolor = \"aliceblue\",\n      color = \"gray70\",\n      fontcolor = \"gray50\"]\n\nedge [fontname = \"Helvetica\",\n     fontsize = \"8\",\n     len = \"1.5\",\n     color = \"gray80\",\n     arrowsize = \"0.5\"]\n\n  \"1\" [label = \"r_oms\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,2!\"] \n  \"2\" [label = \"r_sr\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,3!\"] \n  \"3\" [label = \"r_nb_flo_open\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"1,1!\"] \n  \"4\" [label = \"r_height_max\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"3,1!\"] \n\"3\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.202 \"] \n\"4\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.088 \"] \n\"1\"->\"2\" [fontsize = \"6\", fontsize = \"6\", label = \" 0.352 \"] \n\"3\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.055 \"] \n\"4\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" -0.172 \"] \n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Structural Equation Model of psem_proxy_med 

Call:
  r_oms ~ r_nb_flo_open + r_height_max
  r_sr ~ r_oms + r_nb_flo_open + r_height_max

    AIC
 266.546

---
Tests of directed separation:

 tableau de données (data frame) avec 0 colonnes et 0 ligne

--
Global goodness-of-fit:

Chi-Squared = 0 with P-value = 1 and on 0 degrees of freedom
Fisher's C = NA with P-value = NA and on 0 degrees of freedom

---
Coefficients:

  Response     Predictor Estimate Std.Error      DF Crit.Value P.Value
     r_oms r_nb_flo_open   0.1918    0.1716 32.0055     1.2501  0.2718
     r_oms  r_height_max   0.3267    0.6745 32.0055     0.2345  0.6315
      r_sr         r_oms   0.7720    0.2679 42.7740     8.0197  0.0070
      r_sr r_nb_flo_open   0.1146    0.2421 28.9865     0.2229  0.6404
      r_sr  r_height_max  -1.4075    0.9418 29.2146     2.2152  0.1474
  Std.Estimate   
        0.2023   
        0.0876   
        0.3525 **
        0.0552   
       -0.1724   

  Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05

---
Individual R-squared:

  Response method Marginal Conditional
     r_oms   none     0.04        0.86
      r_sr   none     0.13        0.13</code></pre>
</div>
</div>
<p>For the high treatment :</p>
<div class="cell">
<div class="cell-output-display">
<div class="grViz html-widget html-fill-item" id="htmlwidget-00d65fa4c31d9a5d477d" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-00d65fa4c31d9a5d477d">{"x":{"diagram":"digraph {\n\ngraph [layout = \"neato\",\n       outputorder = \"edgesfirst\",\n       bgcolor = \"white\"]\n\nnode [fontname = \"Helvetica\",\n      fontsize = \"10\",\n      shape = \"circle\",\n      fixedsize = \"true\",\n      width = \"0.5\",\n      style = \"filled\",\n      fillcolor = \"aliceblue\",\n      color = \"gray70\",\n      fontcolor = \"gray50\"]\n\nedge [fontname = \"Helvetica\",\n     fontsize = \"8\",\n     len = \"1.5\",\n     color = \"gray80\",\n     arrowsize = \"0.5\"]\n\n  \"1\" [label = \"r_oms\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,2!\"] \n  \"2\" [label = \"r_sr\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,3!\"] \n  \"3\" [label = \"r_nb_flo_open\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"1,1!\"] \n  \"4\" [label = \"r_height_max\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"3,1!\"] \n\"3\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.421 \"] \n\"4\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.02 \"] \n\"1\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.203 \"] \n\"3\"->\"2\" [fontsize = \"6\", fontsize = \"6\", label = \" 0.705 \"] \n\"4\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.025 \"] \n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Structural Equation Model of psem_proxy_high 

Call:
  r_oms ~ r_nb_flo_open + r_height_max
  r_sr ~ r_oms + r_nb_flo_open + r_height_max

    AIC
 29.888

---
Tests of directed separation:

 tableau de données (data frame) avec 0 colonnes et 0 ligne

--
Global goodness-of-fit:

Chi-Squared = 0 with P-value = 1 and on 0 degrees of freedom
Fisher's C = NA with P-value = NA and on 0 degrees of freedom

---
Coefficients:

  Response     Predictor Estimate Std.Error      DF Crit.Value P.Value
     r_oms r_nb_flo_open   0.1098    0.1246  7.0000     0.7770  0.4073
     r_oms  r_height_max   0.0230    0.5572  7.0000     0.0017  0.9682
      r_sr         r_oms   1.0067    1.0964 10.1902     0.6968  0.4230
      r_sr r_nb_flo_open   0.9132    0.3706  8.1699     5.8224  0.0417
      r_sr  r_height_max   0.1439    1.6453  8.1560     0.0073  0.9338
  Std.Estimate  
        0.4207  
        0.0197  
        0.2030  
        0.7052 *
        0.0248  

  Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05

---
Individual R-squared:

  Response method Marginal Conditional
     r_oms   none     0.16        0.71
      r_sr   none     0.59        0.59</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Strongest effect for the height, with a negative effect of maximum height and a positive effect for mean height on the number of contacts, and the opposite effects for the observational mating success</strong>.</p>
</blockquote>
</section>
<section id="all-sessions-with-sampled-genotypes" class="level4">
<h4 class="anchored" data-anchor-id="all-sessions-with-sampled-genotypes">All sessions with sampled genotypes</h4>
<p>For the low treatment :</p>
<div class="cell">
<div class="cell-output-display">
<div class="grViz html-widget html-fill-item" id="htmlwidget-c7f948e89b069960abf0" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-c7f948e89b069960abf0">{"x":{"diagram":"digraph {\n\ngraph [layout = \"neato\",\n       outputorder = \"edgesfirst\",\n       bgcolor = \"white\"]\n\nnode [fontname = \"Helvetica\",\n      fontsize = \"10\",\n      shape = \"circle\",\n      fixedsize = \"true\",\n      width = \"0.5\",\n      style = \"filled\",\n      fillcolor = \"aliceblue\",\n      color = \"gray70\",\n      fontcolor = \"gray50\"]\n\nedge [fontname = \"Helvetica\",\n     fontsize = \"8\",\n     len = \"1.5\",\n     color = \"gray80\",\n     arrowsize = \"0.5\"]\n\n  \"1\" [label = \"r_oms\", fillcolor = \"#FFE4C4\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,2!\"] \n  \"2\" [label = \"r_sr\", fillcolor = \"#FFE4C4\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,3!\"] \n  \"3\" [label = \"r_nb_flo_open\", fillcolor = \"#FFE4C4\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"1,1!\"] \n  \"4\" [label = \"r_height_max\", fillcolor = \"#FFE4C4\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"3,1!\"] \n\"3\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" -0.034 \"] \n\"4\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" -0.074 \"] \n\"1\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.059 \"] \n\"3\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.149 \"] \n\"4\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.038 \"] \n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Structural Equation Model of psem_proxy_med 

Call:
  r_oms ~ r_nb_flo_open + r_height_max
  r_sr ~ r_oms + r_nb_flo_open + r_height_max

    AIC
 466.401

---
Tests of directed separation:

 tableau de données (data frame) avec 0 colonnes et 0 ligne

--
Global goodness-of-fit:

Chi-Squared = 0 with P-value = 1 and on 0 degrees of freedom
Fisher's C = NA with P-value = NA and on 0 degrees of freedom

---
Coefficients:

  Response     Predictor Estimate Std.Error      DF Crit.Value P.Value
     r_oms r_nb_flo_open  -0.0862    0.3024 52.0000     0.0812  0.7767
     r_oms  r_height_max  -0.0540    0.0882 52.0000     0.3757  0.5426
      r_sr         r_oms   0.0521    0.1230 55.5873     0.1637  0.6873
      r_sr r_nb_flo_open   0.3301    0.3043 44.8655     1.1400  0.2914
      r_sr  r_height_max   0.0248    0.0825 67.2553     0.0886  0.7669
  Std.Estimate 
       -0.0343 
       -0.0738 
        0.0592 
        0.1493 
        0.0385 

  Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05

---
Individual R-squared:

  Response method Marginal Conditional
     r_oms   none     0.01        0.67
      r_sr   none     0.02        0.02</code></pre>
</div>
</div>
<p>For the medium treatment :</p>
<div class="cell">
<div class="cell-output-display">
<div class="grViz html-widget html-fill-item" id="htmlwidget-6a1beb9a4ab9fa2c7b3f" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-6a1beb9a4ab9fa2c7b3f">{"x":{"diagram":"digraph {\n\ngraph [layout = \"neato\",\n       outputorder = \"edgesfirst\",\n       bgcolor = \"white\"]\n\nnode [fontname = \"Helvetica\",\n      fontsize = \"10\",\n      shape = \"circle\",\n      fixedsize = \"true\",\n      width = \"0.5\",\n      style = \"filled\",\n      fillcolor = \"aliceblue\",\n      color = \"gray70\",\n      fontcolor = \"gray50\"]\n\nedge [fontname = \"Helvetica\",\n     fontsize = \"8\",\n     len = \"1.5\",\n     color = \"gray80\",\n     arrowsize = \"0.5\"]\n\n  \"1\" [label = \"r_oms\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,2!\"] \n  \"2\" [label = \"r_sr\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,3!\"] \n  \"3\" [label = \"r_nb_flo_open\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"1,1!\"] \n  \"4\" [label = \"r_height_max\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"3,1!\"] \n\"3\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.193 \"] \n\"4\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.168 \"] \n\"1\"->\"2\" [fontsize = \"6\", fontsize = \"6\", label = \" 0.365 \"] \n\"3\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.101 \"] \n\"4\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" -0.155 \"] \n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Structural Equation Model of psem_proxy_med 

Call:
  r_oms ~ r_nb_flo_open + r_height_max
  r_sr ~ r_oms + r_nb_flo_open + r_height_max

    AIC
 346.555

---
Tests of directed separation:

 tableau de données (data frame) avec 0 colonnes et 0 ligne

--
Global goodness-of-fit:

Chi-Squared = 0 with P-value = 1 and on 0 degrees of freedom
Fisher's C = NA with P-value = NA and on 0 degrees of freedom

---
Coefficients:

  Response     Predictor Estimate Std.Error      DF Crit.Value P.Value
     r_oms r_nb_flo_open   0.2097    0.1482 52.0000     2.0027  0.1630
     r_oms  r_height_max   0.6384    0.5195 52.0000     1.5106  0.2246
      r_sr         r_oms   0.6812    0.2842 69.9725     5.2217  0.0253
      r_sr r_nb_flo_open   0.2051    0.2160 42.5914     0.8817  0.3530
      r_sr  r_height_max  -1.1022    0.7502 40.7087     2.1135  0.1537
  Std.Estimate  
        0.1932  
        0.1678  
        0.3651 *
        0.1013  
       -0.1552  

  Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05

---
Individual R-squared:

  Response method Marginal Conditional
     r_oms   none     0.06        0.84
      r_sr   none     0.10        0.10</code></pre>
</div>
</div>
<p>For the high treatment :</p>
<div class="cell">
<div class="cell-output-display">
<div class="grViz html-widget html-fill-item" id="htmlwidget-ebee00a4711fcdc95662" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-ebee00a4711fcdc95662">{"x":{"diagram":"digraph {\n\ngraph [layout = \"neato\",\n       outputorder = \"edgesfirst\",\n       bgcolor = \"white\"]\n\nnode [fontname = \"Helvetica\",\n      fontsize = \"10\",\n      shape = \"circle\",\n      fixedsize = \"true\",\n      width = \"0.5\",\n      style = \"filled\",\n      fillcolor = \"aliceblue\",\n      color = \"gray70\",\n      fontcolor = \"gray50\"]\n\nedge [fontname = \"Helvetica\",\n     fontsize = \"8\",\n     len = \"1.5\",\n     color = \"gray80\",\n     arrowsize = \"0.5\"]\n\n  \"1\" [label = \"r_oms\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,2!\"] \n  \"2\" [label = \"r_sr\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,3!\"] \n  \"3\" [label = \"r_nb_flo_open\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"1,1!\"] \n  \"4\" [label = \"r_height_max\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"3,1!\"] \n\"3\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.144 \"] \n\"4\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.05 \"] \n\"1\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.179 \"] \n\"3\"->\"2\" [fontsize = \"6\", fontsize = \"6\", label = \" 0.412 \"] \n\"4\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" -0.013 \"] \n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Structural Equation Model of psem_proxy_high 

Call:
  r_oms ~ r_nb_flo_open + r_height_max
  r_sr ~ r_oms + r_nb_flo_open + r_height_max

    AIC
 99.066

---
Tests of directed separation:

 tableau de données (data frame) avec 0 colonnes et 0 ligne

--
Global goodness-of-fit:

Chi-Squared = 0 with P-value = 1 and on 0 degrees of freedom
Fisher's C = NA with P-value = NA and on 0 degrees of freedom

---
Coefficients:

  Response     Predictor Estimate Std.Error      DF Crit.Value P.Value
     r_oms r_nb_flo_open   0.0484    0.0540 37.0000     0.8026  0.3761
     r_oms  r_height_max   0.0494    0.1594 37.0000     0.0959  0.7585
      r_sr         r_oms   0.9535    0.7028 57.5173     1.7504  0.1911
      r_sr r_nb_flo_open   0.7407    0.2516 30.8012     8.6023  0.0063
      r_sr  r_height_max  -0.0660    0.7256 33.8325     0.0082  0.9284
  Std.Estimate   
        0.1436   
        0.0496   
        0.1790   
        0.4124 **
       -0.0125   

  Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05

---
Individual R-squared:

  Response method Marginal Conditional
     r_oms   none     0.03        0.44
      r_sr   none     0.19        0.19</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p><strong>Strongest effect for the height, with a negative effect of maximum height and a positive effect for mean height on the number of contacts, and the opposite effects for the observational mating success</strong>.</p>
</blockquote>
</section>
<section id="per-sex-all-sessions-with-sampled-genotypes" class="level4">
<h4 class="anchored" data-anchor-id="per-sex-all-sessions-with-sampled-genotypes">Per sex: all sessions with sampled genotypes</h4>
<p><strong>For the low treatment :</strong></p>
<div class="grid">
<div class="g-col-6">
<p>Females:</p>
<div class="cell">
<div class="cell-output-display">
<div class="grViz html-widget html-fill-item" id="htmlwidget-f58f7a58896e650fa0db" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-f58f7a58896e650fa0db">{"x":{"diagram":"digraph {\n\ngraph [layout = \"neato\",\n       outputorder = \"edgesfirst\",\n       bgcolor = \"white\"]\n\nnode [fontname = \"Helvetica\",\n      fontsize = \"10\",\n      shape = \"circle\",\n      fixedsize = \"true\",\n      width = \"0.5\",\n      style = \"filled\",\n      fillcolor = \"aliceblue\",\n      color = \"gray70\",\n      fontcolor = \"gray50\"]\n\nedge [fontname = \"Helvetica\",\n     fontsize = \"8\",\n     len = \"1.5\",\n     color = \"gray80\",\n     arrowsize = \"0.5\"]\n\n  \"1\" [label = \"r_oms\", fillcolor = \"#FFE4C4\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,2!\"] \n  \"2\" [label = \"r_sr\", fillcolor = \"#FFE4C4\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,3!\"] \n  \"3\" [label = \"r_nb_flo_open\", fillcolor = \"#FFE4C4\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"1,1!\"] \n  \"4\" [label = \"r_height_max\", fillcolor = \"#FFE4C4\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"3,1!\"] \n\"3\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.051 \"] \n\"4\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.02 \"] \n\"1\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.052 \"] \n\"3\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.133 \"] \n\"4\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.042 \"] \n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Structural Equation Model of psem_proxy_med 

Call:
  r_oms ~ r_nb_flo_open + r_height_max
  r_sr ~ r_oms + r_nb_flo_open + r_height_max

    AIC
 269.970

---
Tests of directed separation:

 tableau de données (data frame) avec 0 colonnes et 0 ligne

--
Global goodness-of-fit:

Chi-Squared = 0 with P-value = 1 and on 0 degrees of freedom
Fisher's C = NA with P-value = NA and on 0 degrees of freedom

---
Coefficients:

  Response     Predictor Estimate Std.Error      DF Crit.Value P.Value
     r_oms r_nb_flo_open   0.1264    0.3264 52.0000     0.1499  0.7002
     r_oms  r_height_max   0.0145    0.0952 52.0000     0.0231  0.8797
      r_sr         r_oms   0.0545    0.2317 29.4955     0.0491  0.8262
      r_sr r_nb_flo_open   0.3447    0.5987 32.1006     0.3195  0.5758
      r_sr  r_height_max   0.0314    0.1047 30.1815     0.0885  0.7681
  Std.Estimate 
        0.0513 
        0.0201 
        0.0519 
        0.1331 
        0.0416 

  Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05

---
Individual R-squared:

  Response method Marginal Conditional
     r_oms   none     0.00        0.00
      r_sr   none     0.01        0.01</code></pre>
</div>
</div>
</div>
<div class="g-col-6">
<p>Males:</p>
<div class="cell">
<div class="cell-output-display">
<div class="grViz html-widget html-fill-item" id="htmlwidget-23b6f28ff00a6ef24fd9" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-23b6f28ff00a6ef24fd9">{"x":{"diagram":"digraph {\n\ngraph [layout = \"neato\",\n       outputorder = \"edgesfirst\",\n       bgcolor = \"white\"]\n\nnode [fontname = \"Helvetica\",\n      fontsize = \"10\",\n      shape = \"circle\",\n      fixedsize = \"true\",\n      width = \"0.5\",\n      style = \"filled\",\n      fillcolor = \"aliceblue\",\n      color = \"gray70\",\n      fontcolor = \"gray50\"]\n\nedge [fontname = \"Helvetica\",\n     fontsize = \"8\",\n     len = \"1.5\",\n     color = \"gray80\",\n     arrowsize = \"0.5\"]\n\n  \"1\" [label = \"r_oms\", fillcolor = \"#FFE4C4\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,2!\"] \n  \"2\" [label = \"r_sr\", fillcolor = \"#FFE4C4\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,3!\"] \n  \"3\" [label = \"r_nb_flo_open\", fillcolor = \"#FFE4C4\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"1,1!\"] \n  \"4\" [label = \"r_height_max\", fillcolor = \"#FFE4C4\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"3,1!\"] \n\"3\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" -0.117 \"] \n\"4\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" -0.164 \"] \n\"1\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.072 \"] \n\"3\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.181 \"] \n\"4\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" -0.092 \"] \n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Structural Equation Model of psem_proxy_med 

Call:
  r_oms ~ r_nb_flo_open + r_height_max
  r_sr ~ r_oms + r_nb_flo_open + r_height_max

    AIC
 249.214

---
Tests of directed separation:

 tableau de données (data frame) avec 0 colonnes et 0 ligne

--
Global goodness-of-fit:

Chi-Squared = 0 with P-value = 1 and on 0 degrees of freedom
Fisher's C = NA with P-value = NA and on 0 degrees of freedom

---
Coefficients:

  Response     Predictor Estimate Std.Error      DF Crit.Value P.Value
     r_oms r_nb_flo_open  -0.2988    0.3327 52.0000     0.8063  0.3734
     r_oms  r_height_max  -0.1226    0.0970 52.0000     1.5963  0.2121
      r_sr         r_oms   0.0505    0.1327 34.9846     0.1389  0.7116
      r_sr r_nb_flo_open   0.3252    0.3186 33.3045     1.0330  0.3168
      r_sr  r_height_max  -0.0482    0.2531  3.1420     0.0323  0.8684
  Std.Estimate 
       -0.1167 
       -0.1642 
        0.0720 
        0.1813 
       -0.0921 

  Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05

---
Individual R-squared:

  Response method Marginal Conditional
     r_oms   none     0.04        0.04
      r_sr   none     0.03        0.03</code></pre>
</div>
</div>
</div>
</div>
<p><strong>For the medium treatment :</strong></p>
<div class="grid">
<div class="g-col-6">
<p>Females:</p>
<div class="cell">
<div class="cell-output-display">
<div class="grViz html-widget html-fill-item" id="htmlwidget-929051d272056d581eb7" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-929051d272056d581eb7">{"x":{"diagram":"digraph {\n\ngraph [layout = \"neato\",\n       outputorder = \"edgesfirst\",\n       bgcolor = \"white\"]\n\nnode [fontname = \"Helvetica\",\n      fontsize = \"10\",\n      shape = \"circle\",\n      fixedsize = \"true\",\n      width = \"0.5\",\n      style = \"filled\",\n      fillcolor = \"aliceblue\",\n      color = \"gray70\",\n      fontcolor = \"gray50\"]\n\nedge [fontname = \"Helvetica\",\n     fontsize = \"8\",\n     len = \"1.5\",\n     color = \"gray80\",\n     arrowsize = \"0.5\"]\n\n  \"1\" [label = \"r_oms\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,2!\"] \n  \"2\" [label = \"r_sr\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,3!\"] \n  \"3\" [label = \"r_nb_flo_open\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"1,1!\"] \n  \"4\" [label = \"r_height_max\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"3,1!\"] \n\"3\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.22 \"] \n\"4\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.103 \"] \n\"1\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.222 \"] \n\"3\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.085 \"] \n\"4\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" -0.206 \"] \n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Structural Equation Model of psem_proxy_med 

Call:
  r_oms ~ r_nb_flo_open + r_height_max
  r_sr ~ r_oms + r_nb_flo_open + r_height_max

    AIC
 206.342

---
Tests of directed separation:

 tableau de données (data frame) avec 0 colonnes et 0 ligne

--
Global goodness-of-fit:

Chi-Squared = 0 with P-value = 1 and on 0 degrees of freedom
Fisher's C = NA with P-value = NA and on 0 degrees of freedom

---
Coefficients:

  Response     Predictor Estimate Std.Error      DF Crit.Value P.Value
     r_oms r_nb_flo_open   0.2451    0.1444 52.0000     2.8827  0.0955
     r_oms  r_height_max   0.4028    0.5061 52.0000     0.6334  0.4297
      r_sr         r_oms   0.3979    0.4175 36.9129     0.8621  0.3592
      r_sr r_nb_flo_open   0.1697    0.3271 33.8476     0.2688  0.6075
      r_sr  r_height_max  -1.4474    1.0410 35.1121     1.9060  0.1761
  Std.Estimate 
        0.2197 
        0.1030 
        0.2215 
        0.0847 
       -0.2060 

  Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05

---
Individual R-squared:

  Response method Marginal Conditional
     r_oms   none     0.06        0.06
      r_sr   none     0.08        0.08</code></pre>
</div>
</div>
</div>
<div class="g-col-6">
<p>Males:</p>
<div class="cell">
<div class="cell-output-display">
<div class="grViz html-widget html-fill-item" id="htmlwidget-42c32d868aa39a74d71f" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-42c32d868aa39a74d71f">{"x":{"diagram":"digraph {\n\ngraph [layout = \"neato\",\n       outputorder = \"edgesfirst\",\n       bgcolor = \"white\"]\n\nnode [fontname = \"Helvetica\",\n      fontsize = \"10\",\n      shape = \"circle\",\n      fixedsize = \"true\",\n      width = \"0.5\",\n      style = \"filled\",\n      fillcolor = \"aliceblue\",\n      color = \"gray70\",\n      fontcolor = \"gray50\"]\n\nedge [fontname = \"Helvetica\",\n     fontsize = \"8\",\n     len = \"1.5\",\n     color = \"gray80\",\n     arrowsize = \"0.5\"]\n\n  \"1\" [label = \"r_oms\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,2!\"] \n  \"2\" [label = \"r_sr\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,3!\"] \n  \"3\" [label = \"r_nb_flo_open\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"1,1!\"] \n  \"4\" [label = \"r_height_max\", fillcolor = \"#FFA500\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"3,1!\"] \n\"3\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.165 \"] \n\"4\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.236 \"] \n\"1\"->\"2\" [fontsize = \"6\", fontsize = \"6\", label = \" 0.483 \"] \n\"3\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.105 \"] \n\"4\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" -0.12 \"] \n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Structural Equation Model of psem_proxy_med 

Call:
  r_oms ~ r_nb_flo_open + r_height_max
  r_sr ~ r_oms + r_nb_flo_open + r_height_max

    AIC
 201.686

---
Tests of directed separation:

 tableau de données (data frame) avec 0 colonnes et 0 ligne

--
Global goodness-of-fit:

Chi-Squared = 0 with P-value = 1 and on 0 degrees of freedom
Fisher's C = NA with P-value = NA and on 0 degrees of freedom

---
Coefficients:

  Response     Predictor Estimate Std.Error      DF Crit.Value P.Value
     r_oms r_nb_flo_open   0.1742    0.1343 52.0000     1.6839  0.2001
     r_oms  r_height_max   0.8741    0.4708 52.0000     3.4472  0.0690
      r_sr         r_oms   0.9400    0.4047 34.4543     4.7470  0.0363
      r_sr r_nb_flo_open   0.2154    0.2982 39.6650     0.4988  0.4841
      r_sr  r_height_max  -0.8627    1.1393 38.4220     0.5553  0.4607
  Std.Estimate  
        0.1653  
        0.2365  
        0.4830 *
        0.1050  
       -0.1199  

  Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05

---
Individual R-squared:

  Response method Marginal Conditional
     r_oms   none     0.09        0.09
      r_sr   none     0.14        0.14</code></pre>
</div>
</div>
</div>
</div>
<p><strong>For the high treatment :</strong></p>
<div class="grid">
<div class="g-col-6">
<p>Females:</p>
<div class="cell">
<div class="cell-output-display">
<div class="grViz html-widget html-fill-item" id="htmlwidget-53f4f834cd6d1748c5ba" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-53f4f834cd6d1748c5ba">{"x":{"diagram":"digraph {\n\ngraph [layout = \"neato\",\n       outputorder = \"edgesfirst\",\n       bgcolor = \"white\"]\n\nnode [fontname = \"Helvetica\",\n      fontsize = \"10\",\n      shape = \"circle\",\n      fixedsize = \"true\",\n      width = \"0.5\",\n      style = \"filled\",\n      fillcolor = \"aliceblue\",\n      color = \"gray70\",\n      fontcolor = \"gray50\"]\n\nedge [fontname = \"Helvetica\",\n     fontsize = \"8\",\n     len = \"1.5\",\n     color = \"gray80\",\n     arrowsize = \"0.5\"]\n\n  \"1\" [label = \"r_oms\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,2!\"] \n  \"2\" [label = \"r_sr\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,3!\"] \n  \"3\" [label = \"r_nb_flo_open\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"1,1!\"] \n  \"4\" [label = \"r_height_max\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"3,1!\"] \n\"3\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.177 \"] \n\"4\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.01 \"] \n\"1\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.186 \"] \n\"3\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.292 \"] \n\"4\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.236 \"] \n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Structural Equation Model of psem_proxy_med 

Call:
  r_oms ~ r_nb_flo_open + r_height_max
  r_sr ~ r_oms + r_nb_flo_open + r_height_max

    AIC
 71.547

---
Tests of directed separation:

 tableau de données (data frame) avec 0 colonnes et 0 ligne

--
Global goodness-of-fit:

Chi-Squared = 0 with P-value = 1 and on 0 degrees of freedom
Fisher's C = NA with P-value = NA and on 0 degrees of freedom

---
Coefficients:

  Response     Predictor Estimate Std.Error      DF Crit.Value P.Value
     r_oms r_nb_flo_open   0.0657    0.0714 34.0000     0.8480  0.3636
     r_oms  r_height_max   0.0113    0.2106 34.0000     0.0029  0.9574
      r_sr         r_oms   0.8686    0.7812 28.7681     1.1750  0.2874
      r_sr r_nb_flo_open   0.5073    0.3357 29.1498     2.1399  0.1542
      r_sr  r_height_max   1.2087    1.0369 29.3296     1.2630  0.2702
  Std.Estimate 
        0.1768 
        0.0103 
        0.1862 
        0.2925 
        0.2362 

  Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05

---
Individual R-squared:

  Response method Marginal Conditional
     r_oms   none     0.03        0.03
      r_sr   none     0.25        0.25</code></pre>
</div>
</div>
</div>
<div class="g-col-6">
<p>Males:</p>
<div class="cell">
<div class="cell-output-display">
<div class="grViz html-widget html-fill-item" id="htmlwidget-18dbd9d7ca93b09745c9" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-18dbd9d7ca93b09745c9">{"x":{"diagram":"digraph {\n\ngraph [layout = \"neato\",\n       outputorder = \"edgesfirst\",\n       bgcolor = \"white\"]\n\nnode [fontname = \"Helvetica\",\n      fontsize = \"10\",\n      shape = \"circle\",\n      fixedsize = \"true\",\n      width = \"0.5\",\n      style = \"filled\",\n      fillcolor = \"aliceblue\",\n      color = \"gray70\",\n      fontcolor = \"gray50\"]\n\nedge [fontname = \"Helvetica\",\n     fontsize = \"8\",\n     len = \"1.5\",\n     color = \"gray80\",\n     arrowsize = \"0.5\"]\n\n  \"1\" [label = \"r_oms\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,2!\"] \n  \"2\" [label = \"r_sr\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"2,3!\"] \n  \"3\" [label = \"r_nb_flo_open\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"1,1!\"] \n  \"4\" [label = \"r_height_max\", fillcolor = \"#8B5A00\", fontsize = \"6\", fontcolor = \"#FFFFFF\", pos = \"3,1!\"] \n\"3\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.104 \"] \n\"4\"->\"1\" [fontsize = \"6\", style = \"dashed\", label = \" 0.099 \"] \n\"1\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" 0.189 \"] \n\"3\"->\"2\" [fontsize = \"6\", fontsize = \"6\", label = \" 0.5 \"] \n\"4\"->\"2\" [fontsize = \"6\", style = \"dashed\", label = \" -0.198 \"] \n}","config":{"engine":"dot","options":null}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Structural Equation Model of psem_proxy_med 

Call:
  r_oms ~ r_nb_flo_open + r_height_max
  r_sr ~ r_oms + r_nb_flo_open + r_height_max

    AIC
 57.356

---
Tests of directed separation:

 tableau de données (data frame) avec 0 colonnes et 0 ligne

--
Global goodness-of-fit:

Chi-Squared = 0 with P-value = 1 and on 0 degrees of freedom
Fisher's C = NA with P-value = NA and on 0 degrees of freedom

---
Coefficients:

  Response     Predictor Estimate Std.Error      DF Crit.Value P.Value
     r_oms r_nb_flo_open   0.0311    0.0574 34.0000     0.2938  0.5914
     r_oms  r_height_max   0.0874    0.1692 34.0000     0.2669  0.6088
      r_sr         r_oms   1.1823    1.4745 27.6486     0.5956  0.4468
      r_sr r_nb_flo_open   0.9318    0.3891 25.2728     5.6997  0.0248
      r_sr  r_height_max  -1.0884    1.0558 25.2078     1.0586  0.3133
  Std.Estimate  
        0.1041  
        0.0993  
        0.1893  
        0.4998 *
       -0.1979  

  Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05

---
Individual R-squared:

  Response method Marginal Conditional
     r_oms   none     0.03        0.03
      r_sr   none     0.17        0.17</code></pre>
</div>
</div>
</div>
</div>
</section>
</section>
</section>
</section>
<section id="comparing-observational-to-genetic-mating-success" class="level1">
<h1>Comparing observational to genetic mating success</h1>
<section id="true-batemans-gradient-on-all-data-based-on-observations" class="level2">
<h2 class="anchored" data-anchor-id="true-batemans-gradient-on-all-data-based-on-observations">True Bateman’s gradient on all data based on observations</h2>
<p>We stay consistent with the manuscript and work with outcrossed reproductive success,</p>
<p>estimated on all genotypes, and the ratio is then multiplied by the number of total seeds produced per female,</p>
<p>both for the female and the male function (i.e.&nbsp;paternity share).</p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-91-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="bateman-hedges" class="level2">
<h2 class="anchored" data-anchor-id="bateman-hedges">Bateman Hedge’s</h2>
<p>Difference between Bateman’s estimates with genetic sampling versus true Bateman with observational data (sampling gms - true oms). An Hedges’ g &gt; 0 mean that the Bateman gradient estimated with sampling gms is greater than the real ones with true oms.</p>
<p>Observational versus genetic, according to the sampling percentage of genotyped seeds.</p>
<section id="per-session" class="level3">
<h3 class="anchored" data-anchor-id="per-session">Per session:</h3>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-92-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="overall" class="level3">
<h3 class="anchored" data-anchor-id="overall">Overall:</h3>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/unnamed-chunk-93-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="variance-decomposition" class="level2">
<h2 class="anchored" data-anchor-id="variance-decomposition">Variance decomposition</h2>
<p>Observational (with and without flower) versus genetic, according to the sampling percentage of genotyped seeds</p>

</section>
</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body" role="list">
<div id="ref-cresswell_method_1994" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Cresswell JE. 1994 A method for quantifying the gene flow that results from a single bumblebee visit using transgenic oilseed rape,<em><span>Brassica</span> napus</em> <span>L</span>. Cv. <span>Westar</span>. <em>Transgenic Research</em> <strong>3</strong>, 134–137. (doi:<a href="https://doi.org/10.1007/BF01974092">10.1007/BF01974092</a>)</div>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>